

<div class='row' id="top">





<section class="col-md-4">
    <aside class="well">
        <a href="index.html"><h4><i class='fa fa-chevron-left'></i> Back to Tool Docs Index</h4></a>
    </aside>
    <aside class="well">
        <h2>Categories</h2>

        <ul class="nav nav-pills nav-stacked" id="sidenav">
        		<hr>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#EngineParametersavailabletoalltools">Engine Parameters (available to all tools)</a>
							<div id="EngineParametersavailabletoalltools"
								
								<ul>
											<li>
												<a href="org_broadinstitute_gatk_engine_CommandLineGATK.html">CommandLineGATK</a>
											</li>
								</ul>
							</div>
						</li>
        		<hr>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#DiagnosticsandQualityControlTools">Diagnostics and Quality Control Tools</a>
							<div id="DiagnosticsandQualityControlTools"
								
								<ul>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_rnaseq_ASEReadCounter.html">ASEReadCounter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_bqsr_AnalyzeCovariates.html">AnalyzeCovariates</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_coverage_CallableLoci.html">CallableLoci</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_CheckPileup.html">CheckPileup</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_coverage_CompareCallableLoci.html">CompareCallableLoci</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_cancer_contamination_ContEst.html">ContEst</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_CountBases.html">CountBases</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_CountIntervals.html">CountIntervals</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_CountLoci.html">CountLoci</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_CountMales.html">CountMales</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_CountRODs.html">CountRODs</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_CountRODsByRef.html">CountRODsByRef</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_CountReadEvents.html">CountReadEvents</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_CountReads.html">CountReads</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_CountTerminusEvent.html">CountTerminusEvent</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_coverage_DepthOfCoverage.html">DepthOfCoverage</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_diagnostics_diagnosetargets_DiagnoseTargets.html">DiagnoseTargets</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_diffengine_DiffObjects.html">DiffObjects</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_diagnostics_ErrorRatePerCycle.html">ErrorRatePerCycle</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_fasta_FastaStats.html">FastaStats</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_diagnostics_FindCoveredIntervals.html">FindCoveredIntervals</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_FlagStat.html">FlagStat</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_coverage_GCContentByInterval.html">GCContentByInterval</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_Pileup.html">Pileup</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_PrintRODs.html">PrintRODs</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_QCRef.html">QCRef</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_diagnostics_missing_QualifyMissingIntervals.html">QualifyMissingIntervals</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_ReadClippingStats.html">ReadClippingStats</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_diagnostics_ReadGroupProperties.html">ReadGroupProperties</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_diagnostics_ReadLengthDistribution.html">ReadLengthDistribution</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_simulatereads_SimulateReadsForVariants.html">SimulateReadsForVariants</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#SequenceDataProcessingTools">Sequence Data Processing Tools</a>
							<div id="SequenceDataProcessingTools"
								
								<ul>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_bqsr_BaseRecalibrator.html">BaseRecalibrator</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_readutils_ClipReads.html">ClipReads</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_indels_IndelRealigner.html">IndelRealigner</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_indels_LeftAlignIndels.html">LeftAlignIndels</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_readutils_PrintReads.html">PrintReads</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_indels_RealignerTargetCreator.html">RealignerTargetCreator</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_rnaseq_SplitNCigarReads.html">SplitNCigarReads</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_readutils_SplitSamFile.html">SplitSamFile</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#VariantDiscoveryTools">Variant Discovery Tools</a>
							<div id="VariantDiscoveryTools"
								
								<ul>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantrecalibration_ApplyRecalibration.html">ApplyRecalibration</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_GenotypeGVCFs.html">GenotypeGVCFs</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_haplotypecaller_HaplotypeCaller.html">HaplotypeCaller</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_cancer_m2_MuTect2.html">MuTect2</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_genotyper_UnifiedGenotyper.html">UnifiedGenotyper</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantrecalibration_VariantRecalibrator.html">VariantRecalibrator</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#VariantEvaluationandManipulationTools">Variant Evaluation and Manipulation Tools</a>
							<div id="VariantEvaluationandManipulationTools"
								
								<ul>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_CalculateGenotypePosteriors.html">CalculateGenotypePosteriors</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_CatVariants.html">CatVariants</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_CombineGVCFs.html">CombineGVCFs</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_CombineVariants.html">CombineVariants</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_GenotypeConcordance.html">GenotypeConcordance</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_haplotypecaller_HaplotypeResolver.html">HaplotypeResolver</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_LeftAlignAndTrimVariants.html">LeftAlignAndTrimVariants</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_phasing_PhaseByTransmission.html">PhaseByTransmission</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_RandomlySplitVariants.html">RandomlySplitVariants</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_phasing_ReadBackedPhasing.html">ReadBackedPhasing</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_RegenotypeVariants.html">RegenotypeVariants</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_SelectHeaders.html">SelectHeaders</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_SelectVariants.html">SelectVariants</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_VariantAnnotator.html">VariantAnnotator</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_varianteval_VariantEval.html">VariantEval</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_filters_VariantFiltration.html">VariantFiltration</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_VariantsToAllelicPrimitives.html">VariantsToAllelicPrimitives</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_VariantsToBinaryPed.html">VariantsToBinaryPed</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_VariantsToTable.html">VariantsToTable</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_VariantsToVCF.html">VariantsToVCF</a>
											</li>
								</ul>
							</div>
						</li>
        		<hr>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#ReferenceUtilities">Reference Utilities</a>
							<div id="ReferenceUtilities"
								
								<ul>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_fasta_FastaAlternateReferenceMaker.html">FastaAlternateReferenceMaker</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_fasta_FastaReferenceMaker.html">FastaReferenceMaker</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#ValidationUtilities">Validation Utilities</a>
							<div id="ValidationUtilities"
								
								<ul>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_validation_GenotypeAndValidate.html">GenotypeAndValidate</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_ValidateVariants.html">ValidateVariants</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_validation_validationsiteselector_ValidationSiteSelector.html">ValidationSiteSelector</a>
											</li>
								</ul>
							</div>
						</li>
        		<hr>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#ReadFilters">Read Filters</a>
							<div id="ReadFilters"
								
								<ul>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_BadCigarFilter.html">BadCigarFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_BadMateFilter.html">BadMateFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_CountingFilteringIterator$CountingReadFilter.html">CountingFilteringIterator.CountingReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_DuplicateReadFilter.html">DuplicateReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_FailsVendorQualityCheckFilter.html">FailsVendorQualityCheckFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_haplotypecaller_HCMappingQualityFilter.html">HCMappingQualityFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_LibraryReadFilter.html">LibraryReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_MalformedReadFilter.html">MalformedReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_MappingQualityFilter.html">MappingQualityFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_MappingQualityUnavailableFilter.html">MappingQualityUnavailableFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_MappingQualityZeroFilter.html">MappingQualityZeroFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_MateSameStrandFilter.html">MateSameStrandFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_MaxInsertSizeFilter.html">MaxInsertSizeFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_MissingReadGroupFilter.html">MissingReadGroupFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_NoOriginalQualityScoresFilter.html">NoOriginalQualityScoresFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_NotPrimaryAlignmentFilter.html">NotPrimaryAlignmentFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_OverclippedReadFilter.html">OverclippedReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_Platform454Filter.html">Platform454Filter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_PlatformFilter.html">PlatformFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_PlatformUnitFilter.html">PlatformUnitFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_ReadGroupBlackListFilter.html">ReadGroupBlackListFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_ReadLengthFilter.html">ReadLengthFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_ReadNameFilter.html">ReadNameFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_ReadStrandFilter.html">ReadStrandFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_ReassignMappingQualityFilter.html">ReassignMappingQualityFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_ReassignOneMappingQualityFilter.html">ReassignOneMappingQualityFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_SampleFilter.html">SampleFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_SingleReadGroupFilter.html">SingleReadGroupFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_UnmappedReadFilter.html">UnmappedReadFilter</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#RODCodecs">ROD Codecs</a>
							<div id="RODCodecs"
								
								<ul>
											<li>
												<a href="org_broadinstitute_gatk_utils_codecs_beagle_BeagleCodec.html">BeagleCodec</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_utils_codecs_table_BedTableCodec.html">BedTableCodec</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_utils_codecs_hapmap_RawHapMapCodec.html">RawHapMapCodec</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_utils_codecs_refseq_RefSeqCodec.html">RefSeqCodec</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_utils_codecs_sampileup_SAMPileupCodec.html">SAMPileupCodec</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_utils_codecs_samread_SAMReadCodec.html">SAMReadCodec</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_utils_codecs_table_TableCodec.html">TableCodec</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#Uncategorized">Uncategorized</a>
							<div id="Uncategorized"
								
								<ul>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_cancer_contamination_AnnotatePopulationAFWalker.html">AnnotatePopulationAFWalker</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_cancer_AssignSomaticStatus.html">AssignSomaticStatus</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#VariantAnnotations">Variant Annotations</a>
							<div id="VariantAnnotations"
								
								<ul>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AS_BaseQualityRankSumTest.html">AS_BaseQualityRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AS_FisherStrand.html">AS_FisherStrand</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AS_InbreedingCoeff.html">AS_InbreedingCoeff</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AS_MappingQualityRankSumTest.html">AS_MappingQualityRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AS_QualByDepth.html">AS_QualByDepth</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AS_RMSMappingQuality.html">AS_RMSMappingQuality</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AS_ReadPosRankSumTest.html">AS_ReadPosRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AS_StrandOddsRatio.html">AS_StrandOddsRatio</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AlleleBalance.html">AlleleBalance</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AlleleBalanceBySample.html">AlleleBalanceBySample</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AlleleCountBySample.html">AlleleCountBySample</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_BaseCounts.html">BaseCounts</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_BaseCountsBySample.html">BaseCountsBySample</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_BaseQualityRankSumTest.html">BaseQualityRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_cancer_BaseQualitySumPerAlleleBySample.html">BaseQualitySumPerAlleleBySample</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_ChromosomeCounts.html">ChromosomeCounts</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_ClippingRankSumTest.html">ClippingRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_Coverage.html">Coverage</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_DepthPerAlleleBySample.html">DepthPerAlleleBySample</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_DepthPerSampleHC.html">DepthPerSampleHC</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_ExcessHet.html">ExcessHet</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_FisherStrand.html">FisherStrand</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_FractionInformativeReads.html">FractionInformativeReads</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_GCContent.html">GCContent</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_GenotypeSummaries.html">GenotypeSummaries</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_HaplotypeScore.html">HaplotypeScore</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_HardyWeinberg.html">HardyWeinberg</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_HomopolymerRun.html">HomopolymerRun</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_InbreedingCoeff.html">InbreedingCoeff</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_LikelihoodRankSumTest.html">LikelihoodRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_LowMQ.html">LowMQ</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_MVLikelihoodRatio.html">MVLikelihoodRatio</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_MappingQualityRankSumTest.html">MappingQualityRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_MappingQualityZero.html">MappingQualityZero</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_MappingQualityZeroBySample.html">MappingQualityZeroBySample</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_NBaseCount.html">NBaseCount</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_cancer_OxoGReadCounts.html">OxoGReadCounts</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_PossibleDeNovo.html">PossibleDeNovo</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_QualByDepth.html">QualByDepth</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_RMSMappingQuality.html">RMSMappingQuality</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_ReadPosRankSumTest.html">ReadPosRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_SampleList.html">SampleList</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_SnpEff.html">SnpEff</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_SpanningDeletions.html">SpanningDeletions</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_StrandAlleleCountsBySample.html">StrandAlleleCountsBySample</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_StrandBiasBySample.html">StrandBiasBySample</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_StrandOddsRatio.html">StrandOddsRatio</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_TandemRepeatAnnotator.html">TandemRepeatAnnotator</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_TransmissionDisequilibriumTest.html">TransmissionDisequilibriumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_VariantType.html">VariantType</a>
											</li>
								</ul>
							</div>
						</li>
        </ul>
    </aside>


</section>

<div class="col-md-8">      <h1><small>Showing docs for version 3.5-0 </small></h1>     <hr />

    <h1>SelectVariants</h1>

    <p class="lead">Select a subset of variants from a larger callset</p>
        <h3>Category
            <small> Variant Evaluation and Manipulation Tools</small>
        </h3>
		<h3>Traversal
			<small>LocusWalker</small>
		</h3>
		<h3>PartitionBy
			<small>LOCUS</small>
		</h3>

	<hr>
    <h2>Overview</h2>
    

 <p>
 Often, a VCF containing many samples and/or variants will need to be subset in order to facilitate certain analyses
 (e.g. comparing and contrasting cases vs. controls; extracting variant or non-variant loci that meet certain
 requirements, displaying just a few samples in a browser like IGV, etc.). SelectVariants can be used for this purpose.
 </p>
 <p>
 There are many different options for selecting subsets of variants from a larger callset:
 <ul>
     <li>Extract one or more samples from a callset based on either a complete sample name or a pattern match.</li>  
     <li>Specify criteria for inclusion that place thresholds on annotation values, e.g. "DP > 1000" (depth of
 coverage greater than 1000x), "AF < 0.25" (sites with allele frequency less than 0.25). These criteria are written 
 as "JEXL expressions", which are documented in the 
 <a href="http://www.broadinstitute.org/gatk/guide/article?id=1255">article about using JEXL expressions</a>.</li>
     <li>Provide concordance or discordance tracks in order to include or exclude variants that are 
 also present in other given callsets.</li>
     <li>Select variants based on criteria like their type 
 (e.g. INDELs only), evidence of mendelian violation, filtering status, allelicity, and so on.</li>
 </ul>
 </p>
 
 <p>There are also several options for recording the original values of certain annotations that are recalculated 
 when a subsetting the new callset, trimming alleles, and so on.</p>
 
 <h3>Input</h3>
 <p>
 A variant call set from which to select a subset.
 </p>

 <h3>Output</h3>
 <p>
 A new VCF file containing the selected subset of variants.
 </p>

 <h3>Usage examples</h3>
 <h4>Select two samples out of a VCF with many samples</h4>
 <pre>
 java -jar GenomeAnalysisTK.jar \
   -T SelectVariants \
   -R reference.fasta \
   -V input.vcf \
   -o output.vcf \
   -sn SAMPLE_A_PARC \
   -sn SAMPLE_B_ACTG
 </pre>

 <h4>Select two samples and any sample that matches a regular expression</h4>
 <pre>
 java -jar GenomeAnalysisTK.jar \
   -T SelectVariants \
   -R reference.fasta \
   -V input.vcf \
   -o output.vcf \
   -sn SAMPLE_1_PARC \
   -sn SAMPLE_1_ACTG \
   -se 'SAMPLE.+PARC'
 </pre>

 <h4>Exclude two samples and any sample that matches a regular expression:</h4>
 <pre>
 java -jar GenomeAnalysisTK.jar \
   -R ref.fasta \
   -T SelectVariants \
   --variant input.vcf \
   -o output.vcf \
   -xl_sn SAMPLE_1_PARC \
   -xl_sn SAMPLE_1_ACTG \
   -xl_se 'SAMPLE.+PARC'
 </pre>

 <h4>Select any sample that matches a regular expression and sites where the QD annotation is more than 10:</h4>
 <pre>
 java -Xmx2g -jar GenomeAnalysisTK.jar \
   -R ref.fasta \
   -T SelectVariants \
   -R reference.fasta \
   -V input.vcf \
   -o output.vcf \
   -se 'SAMPLE.+PARC' \
   -select "QD > 10.0"
 </pre>

 <h4>Select any sample that does not match a regular expression and sites where the QD annotation is more than 10:</h4>
 <pre>
 java  -jar GenomeAnalysisTK.jar \
   -R ref.fasta \
   -T SelectVariants \
   --variant input.vcf \
   -o output.vcf \
   -se 'SAMPLE.+PARC' \
   -select "QD > 10.0"
   -invertSelect
 </pre>

 <h4>Select a sample and exclude non-variant loci and filtered loci (trim remaining alleles by default):</h4>
 <pre>
 java -jar GenomeAnalysisTK.jar \
   -R ref.fasta \
   -T SelectVariants \
   -R reference.fasta \
   -V input.vcf \
   -o output.vcf \
   -sn SAMPLE_1_ACTG \
   -env \
   -ef
 </pre>

 <h4>Select a sample, subset remaining alleles, but don't trim:</h4>
 <pre>
 java -jar GenomeAnalysisTK.jar \
   -T SelectVariants \
   -R reference.fasta \
   -V input.vcf \
   -o output.vcf \
   -sn SAMPLE_1_ACTG \
   -env \
   -noTrim
</pre>

 <h4>Select a sample and restrict the output vcf to a set of intervals:</h4>
 <pre>
 java -jar GenomeAnalysisTK.jar \
   -T SelectVariants \
   -R reference.fasta \
   -V input.vcf \
   -o output.vcf \
   -L /path/to/my.interval_list \
   -sn SAMPLE_1_ACTG
 </pre>

 <h4>Select all calls missed in my vcf, but present in HapMap (useful to take a look at why these variants weren't called in my dataset):</h4>
 <pre>
 java -jar GenomeAnalysisTK.jar \
   -T SelectVariants \
   -R reference.fasta \
   -V hapmap.vcf \
   --discordance myCalls.vcf \
   -o output.vcf \
   -sn mySample
 </pre>

 <h4>Select all calls made by both myCalls and theirCalls (useful to take a look at what is consistent between two callers):</h4>
 <pre>
 java -jar GenomeAnalysisTK.jar \
   -T SelectVariants \
   -R reference.fasta \
   -V myCalls.vcf \
   --concordance theirCalls.vcf \
   -o output.vcf \
   -sn mySample
 </pre>

 <h4>Generating a VCF of all the variants that are mendelian violations. The optional argument '-mvq' restricts the selection to sites that have a QUAL score of 50 or more</h4>
 <pre>
 java -jar GenomeAnalysisTK.jar \
   -T SelectVariants \
   -R reference.fasta \
   -V input.vcf \
   -ped family.ped \
   -mv -mvq 50 \
   -o violations.vcf
 </pre>

 <h4>Generating a VCF of all the variants that are not mendelian violations. The optional argument '-mvq' together with '-invMv' restricts the selection to sites that have a QUAL score of 50 or less</h4>
 <pre>
 java -jar GenomeAnalysisTK.jar \
   -T SelectVariants \
   -R reference.fasta \
   -V input.vcf \
   -ped family.ped \
   -mv -mvq 50 -invMv \
   -o violations.vcf
 </pre>

 <h4>Create a set with 50% of the total number of variants in the variant VCF:</h4>
 <pre>
 java -jar GenomeAnalysisTK.jar \
   -T SelectVariants \
   -R reference.fasta \
   -V input.vcf \
   -o output.vcf \
   -fraction 0.5
 </pre>

 <h4>Select only indels between 2 and 5 bases long from a VCF:</h4>
 <pre>
 java -jar GenomeAnalysisTK.jar \
   -R ref.fasta \
   -T SelectVariants \
   -R reference.fasta \
   -V input.vcf \
   -o output.vcf \
   -selectType INDEL
   --minIndelSize 2
   --maxIndelSize 5
 </pre>

 <h4>Exclude indels from a VCF:</h4>
 <pre>
 java -Xmx2g -jar GenomeAnalysisTK.jar \
   -R ref.fasta \
   -T SelectVariants \
   --variant input.vcf \
   -o output.vcf \
   --selectTypeToExclude INDEL
 </pre>

 <h4>Select only multi-allelic SNPs and MNPs from a VCF (i.e. SNPs with more than one allele listed in the ALT column):</h4>
 <pre>
 java -jar GenomeAnalysisTK.jar \
   -T SelectVariants \
   -R reference.fasta \
   -V input.vcf \
   -o output.vcf \
   -selectType SNP -selectType MNP \
   -restrictAllelesTo MULTIALLELIC
 </pre>

 <h4>Select IDs in fileKeep and exclude IDs in fileExclude:</h4>
 <pre>
 java -jar GenomeAnalysisTK.jar \
   -R ref.fasta \
   -T SelectVariants \
   --variant input.vcf \
   -o output.vcf \
   -IDs fileKeep \
   -excludeIDs fileExclude
 </pre>

 <h4>Select sites where there are between 2 and 5 samples and between 10 and 50 percent of the sample genotypes are filtered:</h4>
 <pre>
 java -jar GenomeAnalysisTK.jar \
   -R ref.fasta \
   -T SelectVariants \
   --variant input.vcf \
   --maxFilteredGenotypes 5
   --minFilteredGenotypes 2
   --maxFractionFilteredGenotypes 0.60
   --minFractionFilteredGenotypes 0.10
 </pre>

  <h4>Set filtered genotypes to no-call (./.):</h4>
 <pre>
 java -jar GenomeAnalysisTK.jar \
   -R ref.fasta \
   -T SelectVariants \
   --variant input.vcf \
   --setFilteredGtToNocall
 </pre>
    
			<hr>
			<h2>Additional Information</h2>
			<p></p>
			<h3>Read filters</h3>
				<p>These Read Filters are automatically applied to the data by the Engine before processing by SelectVariants.</p>
			<ul>
					<li><a href="org_broadinstitute_gatk_engine_filters_MalformedReadFilter.html">MalformedReadFilter</a></li>	
					<li><a href="org_broadinstitute_gatk_engine_filters_BadCigarFilter.html">BadCigarFilter</a></li>	
					<li><a href="org_broadinstitute_gatk_engine_filters_UnmappedReadFilter.html">UnmappedReadFilter</a></li>	
					<li><a href="org_broadinstitute_gatk_engine_filters_NotPrimaryAlignmentFilter.html">NotPrimaryAlignmentFilter</a></li>	
					<li><a href="org_broadinstitute_gatk_engine_filters_FailsVendorQualityCheckFilter.html">FailsVendorQualityCheckFilter</a></li>	
					<li><a href="org_broadinstitute_gatk_engine_filters_DuplicateReadFilter.html">DuplicateReadFilter</a></li>	
			</ul>
			<h3>Parallelism options</h3>
				<p>This tool can be run in multi-threaded mode using this option.</p>
			<ul>
					<li><a href="http://www.broadinstitute.org/gatk/guide/tooldocs/org_broadinstitute_gatk_engine_CommandLineGATK.html#-nt">TreeReducible (-nt)</a></li>
			</ul>
			<hr>
			<h2>Command-line Arguments</h2>
			<p></p>
			<h3>Engine arguments</h3>
			<p>All tools inherit arguments from the GATK Engine' "CommandLineGATK" argument collection, which can be
				used to modify various aspects of the tool's function. For example, the -L argument directs the GATK
				engine to restrict processing to specific genomic intervals; or the -rf argument allows you to apply
				certain read filters to exclude some of the data from the analysis.</p>
			<ul>
					<li><a href="org_broadinstitute_gatk_engine_CommandLineGATK.html">CommandLineGATK</a></li>
			</ul>
   
		
				<h3>SelectVariants specific arguments</h3>
				<p>This table summarizes the command-line arguments that are specific to this tool. For more details on each argument, see the list further down below the table or click on an argument name to jump directly to that entry in the list.</p>
				<table class="table table-striped table-bordered table-condensed">
					<thead>
					<tr>
						<th>Argument name(s)</th>
						<!--<th>Short name</th> -->
						<th>Default value</th>
						<th>Summary</th>
					</tr>
					</thead>
					<tbody>
        <tr>
            <th colspan="4" id="row-divider">Required Inputs</th>
        </tr>
            <tr>
                <td><a href="#--variant">--variant</a><br />
                			&nbsp;<em>-V</em>
                </td>
                <!--<td>RodBinding[VariantContext]</td> -->
                <td>NA</td>
                <td>Input VCF file</td>
            </tr>
        <tr>
            <th colspan="4" id="row-divider">Optional Inputs</th>
        </tr>
            <tr>
                <td><a href="#--concordance">--concordance</a><br />
                			&nbsp;<em>-conc</em>
                </td>
                <!--<td>RodBinding[VariantContext]</td> -->
                <td>NA</td>
                <td>Output variants also called in this comparison track</td>
            </tr>
            <tr>
                <td><a href="#--discordance">--discordance</a><br />
                			&nbsp;<em>-disc</em>
                </td>
                <!--<td>RodBinding[VariantContext]</td> -->
                <td>NA</td>
                <td>Output variants not called in this comparison track</td>
            </tr>
            <tr>
                <td><a href="#--exclude_sample_expressions">--exclude_sample_expressions</a><br />
                			&nbsp;<em>-xl_se</em>
                </td>
                <!--<td>Set[String]</td> -->
                <td>NA</td>
                <td>List of sample expressions to exclude</td>
            </tr>
            <tr>
                <td><a href="#--exclude_sample_file">--exclude_sample_file</a><br />
                			&nbsp;<em>-xl_sf</em>
                </td>
                <!--<td>Set[File]</td> -->
                <td>NA</td>
                <td>List of samples to exclude</td>
            </tr>
            <tr>
                <td><a href="#--sample_file">--sample_file</a><br />
                			&nbsp;<em>-sf</em>
                </td>
                <!--<td>Set[File]</td> -->
                <td>NA</td>
                <td>File containing a list of samples to include</td>
            </tr>
        <tr>
            <th colspan="4" id="row-divider">Optional Outputs</th>
        </tr>
            <tr>
                <td><a href="#--out">--out</a><br />
                			&nbsp;<em>-o</em>
                </td>
                <!--<td>VariantContextWriter</td> -->
                <td>NA</td>
                <td>File to which variants should be written</td>
            </tr>
        <tr>
            <th colspan="4" id="row-divider">Optional Parameters</th>
        </tr>
            <tr>
                <td><a href="#--exclude_sample_name">--exclude_sample_name</a><br />
                			&nbsp;<em>-xl_sn</em>
                </td>
                <!--<td>Set[String]</td> -->
                <td>NA</td>
                <td>Exclude genotypes from this sample</td>
            </tr>
            <tr>
                <td><a href="#--excludeIDs">--excludeIDs</a><br />
                			&nbsp;<em>-xlIDs</em>
                </td>
                <!--<td>File</td> -->
                <td>NA</td>
                <td>List of variant IDs to select</td>
            </tr>
            <tr>
                <td><a href="#--keepIDs">--keepIDs</a><br />
                			&nbsp;<em>-IDs</em>
                </td>
                <!--<td>File</td> -->
                <td>NA</td>
                <td>List of variant IDs to select</td>
            </tr>
            <tr>
                <td><a href="#--maxFilteredGenotypes">--maxFilteredGenotypes</a><br />
                </td>
                <!--<td>int</td> -->
                <td>2147483647</td>
                <td>Maximum number of samples filtered at the genotype level</td>
            </tr>
            <tr>
                <td><a href="#--maxFractionFilteredGenotypes">--maxFractionFilteredGenotypes</a><br />
                </td>
                <!--<td>double</td> -->
                <td>1.0</td>
                <td>Maximum fraction of samples filtered at the genotype level</td>
            </tr>
            <tr>
                <td><a href="#--maxIndelSize">--maxIndelSize</a><br />
                </td>
                <!--<td>int</td> -->
                <td>2147483647</td>
                <td>Maximum size of indels to include</td>
            </tr>
            <tr>
                <td><a href="#--mendelianViolationQualThreshold">--mendelianViolationQualThreshold</a><br />
                			&nbsp;<em>-mvq</em>
                </td>
                <!--<td>double</td> -->
                <td>0.0</td>
                <td>Minimum GQ score for each trio member to accept a site as a violation</td>
            </tr>
            <tr>
                <td><a href="#--minFilteredGenotypes">--minFilteredGenotypes</a><br />
                </td>
                <!--<td>int</td> -->
                <td>0</td>
                <td>Minimum number of samples filtered at the genotype level</td>
            </tr>
            <tr>
                <td><a href="#--minFractionFilteredGenotypes">--minFractionFilteredGenotypes</a><br />
                </td>
                <!--<td>double</td> -->
                <td>0.0</td>
                <td>Maximum fraction of samples filtered at the genotype level</td>
            </tr>
            <tr>
                <td><a href="#--minIndelSize">--minIndelSize</a><br />
                </td>
                <!--<td>int</td> -->
                <td>0</td>
                <td>Minimum size of indels to include</td>
            </tr>
            <tr>
                <td><a href="#--remove_fraction_genotypes">--remove_fraction_genotypes</a><br />
                			&nbsp;<em>-fractionGenotypes</em>
                </td>
                <!--<td>double</td> -->
                <td>0.0</td>
                <td>Select a fraction of genotypes at random from the input and sets them to no-call</td>
            </tr>
            <tr>
                <td><a href="#--restrictAllelesTo">--restrictAllelesTo</a><br />
                </td>
                <!--<td>NumberAlleleRestriction</td> -->
                <td>NA</td>
                <td>Select only variants of a particular allelicity</td>
            </tr>
            <tr>
                <td><a href="#--sample_expressions">--sample_expressions</a><br />
                			&nbsp;<em>-se</em>
                </td>
                <!--<td>Set[String]</td> -->
                <td>NA</td>
                <td>Regular expression to select multiple samples</td>
            </tr>
            <tr>
                <td><a href="#--sample_name">--sample_name</a><br />
                			&nbsp;<em>-sn</em>
                </td>
                <!--<td>Set[String]</td> -->
                <td>NA</td>
                <td>Include genotypes from this sample</td>
            </tr>
            <tr>
                <td><a href="#--select_random_fraction">--select_random_fraction</a><br />
                			&nbsp;<em>-fraction</em>
                </td>
                <!--<td>double</td> -->
                <td>0.0</td>
                <td>Select a fraction of variants at random from the input</td>
            </tr>
            <tr>
                <td><a href="#--selectexpressions">--selectexpressions</a><br />
                			&nbsp;<em>-select</em>
                </td>
                <!--<td>ArrayList[String]</td> -->
                <td>NA</td>
                <td>One or more criteria to use when selecting the data</td>
            </tr>
            <tr>
                <td><a href="#--selectTypeToExclude">--selectTypeToExclude</a><br />
                			&nbsp;<em>-xlSelectType</em>
                </td>
                <!--<td>List[Type]</td> -->
                <td>NA</td>
                <td>Do not select certain type of variants from the input file</td>
            </tr>
            <tr>
                <td><a href="#--selectTypeToInclude">--selectTypeToInclude</a><br />
                			&nbsp;<em>-selectType</em>
                </td>
                <!--<td>List[Type]</td> -->
                <td>NA</td>
                <td>Select only a certain type of variants from the input file</td>
            </tr>
        <tr>
            <th colspan="4" id="row-divider">Optional Flags</th>
        </tr>
            <tr>
                <td><a href="#--excludeFiltered">--excludeFiltered</a><br />
                			&nbsp;<em>-ef</em>
                </td>
                <!--<td>boolean</td> -->
                <td>NA</td>
                <td>Don't include filtered sites</td>
            </tr>
            <tr>
                <td><a href="#--excludeNonVariants">--excludeNonVariants</a><br />
                			&nbsp;<em>-env</em>
                </td>
                <!--<td>boolean</td> -->
                <td>NA</td>
                <td>Don't include non-variant sites</td>
            </tr>
            <tr>
                <td><a href="#--forceValidOutput">--forceValidOutput</a><br />
                </td>
                <!--<td>boolean</td> -->
                <td>NA</td>
                <td>Forces output VCF to be compliant to up-to-date version</td>
            </tr>
            <tr>
                <td><a href="#--invertMendelianViolation">--invertMendelianViolation</a><br />
                			&nbsp;<em>-invMv</em>
                </td>
                <!--<td>Boolean</td> -->
                <td>NA</td>
                <td>Output non-mendelian violation sites only</td>
            </tr>
            <tr>
                <td><a href="#--invertselect">--invertselect</a><br />
                			&nbsp;<em>-invertSelect</em>
                </td>
                <!--<td>boolean</td> -->
                <td>NA</td>
                <td>Invert the selection criteria for -select</td>
            </tr>
            <tr>
                <td><a href="#--keepOriginalAC">--keepOriginalAC</a><br />
                </td>
                <!--<td>boolean</td> -->
                <td>NA</td>
                <td>Store the original AC, AF, and AN values after subsetting</td>
            </tr>
            <tr>
                <td><a href="#--keepOriginalDP">--keepOriginalDP</a><br />
                </td>
                <!--<td>boolean</td> -->
                <td>NA</td>
                <td>Store the original DP value after subsetting</td>
            </tr>
            <tr>
                <td><a href="#--mendelianViolation">--mendelianViolation</a><br />
                			&nbsp;<em>-mv</em>
                </td>
                <!--<td>Boolean</td> -->
                <td>NA</td>
                <td>Output mendelian violation sites only</td>
            </tr>
            <tr>
                <td><a href="#--preserveAlleles">--preserveAlleles</a><br />
                			&nbsp;<em>-noTrim</em>
                </td>
                <!--<td>boolean</td> -->
                <td>NA</td>
                <td>Preserve original alleles, do not trim</td>
            </tr>
            <tr>
                <td><a href="#--removeUnusedAlternates">--removeUnusedAlternates</a><br />
                			&nbsp;<em>-trimAlternates</em>
                </td>
                <!--<td>boolean</td> -->
                <td>NA</td>
                <td>Remove alternate alleles not present in any genotypes</td>
            </tr>
            <tr>
                <td><a href="#--setFilteredGtToNocall">--setFilteredGtToNocall</a><br />
                </td>
                <!--<td>boolean</td> -->
                <td>NA</td>
                <td>Set filtered genotypes to no-call</td>
            </tr>
					</tbody>
				</table>
	
						<h3>Argument details</h3>
						<p>Arguments in this list are specific to this tool. Keep in mind that other arguments are available that are shared with other tools (e.g. command-line GATK arguments); see Inherited arguments above.</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--concordance">--concordance </a>
		 / <small>-conc</small>
	</h3>
	<p class="args">
		<b>Output variants also called in this comparison track</b><br />
		A site is considered concordant if (1) we are not looking for specific samples and there is a variant called
 in both the variant and concordance tracks or (2) every sample present in the variant track is present in the
 concordance track and they have the sample genotype call.
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
		<p>--concordance binds reference ordered data. This argument supports ROD files of the following types: <a href=htsjdk_variant_bcf2_BCF2Codec.html>BCF2</a>, <a href=htsjdk_variant_vcf_VCFCodec.html>VCF</a>, <a href=htsjdk_variant_vcf_VCF3Codec.html>VCF3</a></p>
	<p>
		<span class="label label-info ">RodBinding[VariantContext]</span> 
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--discordance">--discordance </a>
		 / <small>-disc</small>
	</h3>
	<p class="args">
		<b>Output variants not called in this comparison track</b><br />
		A site is considered discordant if there exists some sample in the variant track that has a non-reference genotype
 and either the site isn't present in this track, the sample isn't present in this track,
 or the sample is called reference in this track.
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
		<p>--discordance binds reference ordered data. This argument supports ROD files of the following types: <a href=htsjdk_variant_bcf2_BCF2Codec.html>BCF2</a>, <a href=htsjdk_variant_vcf_VCFCodec.html>VCF</a>, <a href=htsjdk_variant_vcf_VCF3Codec.html>VCF3</a></p>
	<p>
		<span class="label label-info ">RodBinding[VariantContext]</span> 
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--exclude_sample_expressions">--exclude_sample_expressions </a>
		 / <small>-xl_se</small>
	</h3>
	<p class="args">
		<b>List of sample expressions to exclude</b><br />
		Using a regular expression allows you to match multiple sample names that have that pattern in common. Note that sample exclusion takes precedence
 over inclusion, so that if a sample is in both lists it will be excluded. This  argument can be specified multiple times in order to use multiple
 different matching patterns.
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
	<p>
		<span class="label label-info ">Set[String]</span> 
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--exclude_sample_file">--exclude_sample_file </a>
		 / <small>-xl_sf</small>
	</h3>
	<p class="args">
		<b>List of samples to exclude</b><br />
		Sample names should be in a plain text file listing one sample name per line. Note that sample exclusion takes precedence over inclusion, so that
 if a sample is in both lists it will be excluded. This argument can be specified multiple times in order to
 provide multiple sample list files.
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
	<p>
		<span class="label label-info ">Set[File]</span> 
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--exclude_sample_name">--exclude_sample_name </a>
		 / <small>-xl_sn</small>
	</h3>
	<p class="args">
		<b>Exclude genotypes from this sample</b><br />
		Note that sample exclusion takes precedence over inclusion, so that if a sample is in both lists it will be
 excluded. This argument can be specified multiple times in order to provide multiple sample names.
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
	<p>
		<span class="label label-info ">Set[String]</span> 
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--excludeFiltered">--excludeFiltered </a>
		 / <small>-ef</small>
	</h3>
	<p class="args">
		<b>Don't include filtered sites</b><br />
		If this flag is enabled, sites that have been marked as filtered (i.e. have anything other than `.` or `PASS`
 in the FILTER field) will be excluded from the output.
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
	<p>
		<span class="label label-info ">boolean</span> 
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--excludeIDs">--excludeIDs </a>
		 / <small>-xlIDs</small>
	</h3>
	<p class="args">
		<b>List of variant IDs to select</b><br />
		If a file containing a list of IDs is provided to this argument, the tool will not select variants whose ID
 field is present in this list of IDs. The matching is done by exact string matching. The expected file format
 is simply plain text with one ID per line.
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
	<p>
		<span class="label label-info ">File</span> 
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--excludeNonVariants">--excludeNonVariants </a>
		 / <small>-env</small>
	</h3>
	<p class="args">
		<b>Don't include non-variant sites</b><br />
		
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
	<p>
		<span class="label label-info ">boolean</span> 
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--forceValidOutput">--forceValidOutput </a>
		 / <small>NA</small>
	</h3>
	<p class="args">
		<b>Forces output VCF to be compliant to up-to-date version</b><br />
		If this argument is provided, the output will be compliant with the version in the header, however it will also
 cause the tool to run slower than without the argument. Without the argument the header will be compliant with
 the up-to-date version, but the output in the body may not be compliant. If an up-to-date input file is used,
 then the output will also be up-to-date regardless of this argument.
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
	<p>
		<span class="label label-info ">boolean</span> 
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--invertMendelianViolation">--invertMendelianViolation </a>
		 / <small>-invMv</small>
	</h3>
	<p class="args">
		<b>Output non-mendelian violation sites only</b><br />
		If this flag is enabled, this tool will select only variants that do not correspond to a mendelian violation as
 determined on the basis of family structure. Requires passing a pedigree file using the engine-level
 `-ped` argument.
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
	<p>
		<span class="label label-info ">Boolean</span> 
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--invertselect">--invertselect </a>
		 / <small>-invertSelect</small>
	</h3>
	<p class="args">
		<b>Invert the selection criteria for -select</b><br />
		Invert the selection criteria for -select.
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
	<p>
		<span class="label label-info ">boolean</span> 
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--keepIDs">--keepIDs </a>
		 / <small>-IDs</small>
	</h3>
	<p class="args">
		<b>List of variant IDs to select</b><br />
		If a file containing a list of IDs is provided to this argument, the tool will only select variants whose ID
 field is present in this list of IDs. The matching is done by exact string matching. The expected file format
 is simply plain text with one ID per line.
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
	<p>
		<span class="label label-info ">File</span> 
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--keepOriginalAC">--keepOriginalAC </a>
		 / <small>-keepOriginalAC</small>
	</h3>
	<p class="args">
		<b>Store the original AC, AF, and AN values after subsetting</b><br />
		When subsetting a callset, this tool recalculates the AC, AF, and AN values corresponding to the contents of the
 subset. If this flag is enabled, the original values of those annotations will be stored in new annotations called
 AC_Orig, AF_Orig, and AN_Orig.
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
	<p>
		<span class="label label-info ">boolean</span> 
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--keepOriginalDP">--keepOriginalDP </a>
		 / <small>-keepOriginalDP</small>
	</h3>
	<p class="args">
		<b>Store the original DP value after subsetting</b><br />
		When subsetting a callset, this tool recalculates the site-level (INFO field) DP value corresponding to the contents of the
 subset. If this flag is enabled, the original value of the DP annotation will be stored in a new annotation called
 DP_Orig.
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
	<p>
		<span class="label label-info ">boolean</span> 
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--maxFilteredGenotypes">--maxFilteredGenotypes </a>
		 / <small>NA</small>
	</h3>
	<p class="args">
		<b>Maximum number of samples filtered at the genotype level</b><br />
		If this argument is provided, select sites where at most a maximum number of samples are filtered at the genotype level.
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
	<p>
		<span class="label label-info ">int</span> 
			&nbsp;<span class="label label-warning">[ [ -</span>
			&nbsp;<span class="label label-warning"> ] ]</span>
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--maxFractionFilteredGenotypes">--maxFractionFilteredGenotypes </a>
		 / <small>NA</small>
	</h3>
	<p class="args">
		<b>Maximum fraction of samples filtered at the genotype level</b><br />
		If this argument is provided, select sites where a fraction or less of the samples are filtered at the genotype level.
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
	<p>
		<span class="label label-info ">double</span> 
			&nbsp;<span class="label label-warning">[ [ -</span>
			&nbsp;<span class="label label-warning"> ] ]</span>
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--maxIndelSize">--maxIndelSize </a>
		 / <small>NA</small>
	</h3>
	<p class="args">
		<b>Maximum size of indels to include</b><br />
		If this argument is provided, indels that are larger than the specified size will be excluded.
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
	<p>
		<span class="label label-info ">int</span> 
			&nbsp;<span class="label label-warning">[ [ -</span>
			&nbsp;<span class="label label-warning"> ] ]</span>
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--mendelianViolation">--mendelianViolation </a>
		 / <small>-mv</small>
	</h3>
	<p class="args">
		<b>Output mendelian violation sites only</b><br />
		If this flag is enabled, this tool will select only variants that correspond to a mendelian violation as
 determined on the basis of family structure. Requires passing a pedigree file using the engine-level
 `-ped` argument.
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
	<p>
		<span class="label label-info ">Boolean</span> 
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--mendelianViolationQualThreshold">--mendelianViolationQualThreshold </a>
		 / <small>-mvq</small>
	</h3>
	<p class="args">
		<b>Minimum GQ score for each trio member to accept a site as a violation</b><br />
		This argument specifies the genotype quality (GQ) threshold that all members of a trio must have in order
 for a site to be accepted as a mendelian violation. Note that the `-mv` flag must be set for this argument to have an effect.
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
	<p>
		<span class="label label-info ">double</span> 
			&nbsp;<span class="label label-warning">[ [ -</span>
			&nbsp;<span class="label label-warning"> ] ]</span>
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--minFilteredGenotypes">--minFilteredGenotypes </a>
		 / <small>NA</small>
	</h3>
	<p class="args">
		<b>Minimum number of samples filtered at the genotype level</b><br />
		If this argument is provided, select sites where at least a minimum number of samples are filtered at the genotype level.
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
	<p>
		<span class="label label-info ">int</span> 
			&nbsp;<span class="label label-warning">[ [ -</span>
			&nbsp;<span class="label label-warning"> ] ]</span>
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--minFractionFilteredGenotypes">--minFractionFilteredGenotypes </a>
		 / <small>NA</small>
	</h3>
	<p class="args">
		<b>Maximum fraction of samples filtered at the genotype level</b><br />
		If this argument is provided, select sites where a fraction or more of the samples are filtered at the genotype level.
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
	<p>
		<span class="label label-info ">double</span> 
			&nbsp;<span class="label label-warning">[ [ -</span>
			&nbsp;<span class="label label-warning"> ] ]</span>
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--minIndelSize">--minIndelSize </a>
		 / <small>NA</small>
	</h3>
	<p class="args">
		<b>Minimum size of indels to include</b><br />
		If this argument is provided, indels that are smaller than the specified size will be excluded.
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
	<p>
		<span class="label label-info ">int</span> 
			&nbsp;<span class="label label-warning">[ [ -</span>
			&nbsp;<span class="label label-warning"> ] ]</span>
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--out">--out </a>
		 / <small>-o</small>
	</h3>
	<p class="args">
		<b>File to which variants should be written</b><br />
		
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
	<p>
		<span class="label label-info ">VariantContextWriter</span> 
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--preserveAlleles">--preserveAlleles </a>
		 / <small>-noTrim</small>
	</h3>
	<p class="args">
		<b>Preserve original alleles, do not trim</b><br />
		The default behavior of this tool is to remove bases common to all remaining alleles after subsetting
 operations have been completed, leaving only their minimal representation. If this flag is enabled, the original
 alleles will be preserved as recorded in the input VCF.
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
	<p>
		<span class="label label-info ">boolean</span> 
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--remove_fraction_genotypes">--remove_fraction_genotypes </a>
		 / <small>-fractionGenotypes</small>
	</h3>
	<p class="args">
		<b>Select a fraction of genotypes at random from the input and sets them to no-call</b><br />
		The value of this argument should be a number between 0 and 1 specifying the fraction of total variants to be
 randomly selected from the input callset and set to no-call (./). Note that this is done using a probabilistic
 function, so the final result is not guaranteed to carry the exact fraction requested. Can be used for large fractions.
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
	<p>
		<span class="label label-info ">double</span> 
			&nbsp;<span class="label label-warning">[ [ -</span>
			&nbsp;<span class="label label-warning"> ] ]</span>
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--removeUnusedAlternates">--removeUnusedAlternates </a>
		 / <small>-trimAlternates</small>
	</h3>
	<p class="args">
		<b>Remove alternate alleles not present in any genotypes</b><br />
		When this flag is enabled, all alternate alleles that are not present in the (output) samples will be removed.
 Note that this even extends to biallelic SNPs - if the alternate allele is not present in any sample, it will be
 removed and the record will contain a '.' in the ALT column. Note also that sites-only VCFs, by definition, do
 not include the alternate allele in any genotype calls.
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
	<p>
		<span class="label label-info ">boolean</span> 
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--restrictAllelesTo">--restrictAllelesTo </a>
		 / <small>-restrictAllelesTo</small>
	</h3>
	<p class="args">
		<b>Select only variants of a particular allelicity</b><br />
		When this argument is used, we can choose to include only multiallelic or biallelic sites, depending on how many alleles are listed in the ALT column of a VCF.
 For example, a multiallelic record such as:
     1    100 .   A   AAA,AAAAA
 will be excluded if `-restrictAllelesTo BIALLELIC` is used, because there are two alternate alleles, whereas a record such as:
     1    100 .   A  T
 will be included in that case, but would be excluded if `-restrictAllelesTo MULTIALLELIC` is used.
 Valid options are ALL (default), MULTIALLELIC or BIALLELIC.
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
		<p>
		The --restrictAllelesTo argument is an enumerated type (NumberAlleleRestriction), which can have one of the following values:
		<dl class="enum">
				<dt class="enum">ALL</dt>
				<dd class="enum"></dd>
				<dt class="enum">BIALLELIC</dt>
				<dd class="enum"></dd>
				<dt class="enum">MULTIALLELIC</dt>
				<dd class="enum"></dd>
		</dl>
		</p>
	<p>
		<span class="label label-info ">NumberAlleleRestriction</span> 
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--sample_expressions">--sample_expressions </a>
		 / <small>-se</small>
	</h3>
	<p class="args">
		<b>Regular expression to select multiple samples</b><br />
		Using a regular expression allows you to match multiple sample names that have that pattern in common. This
 argument can be specified multiple times in order to use multiple different matching patterns.
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
	<p>
		<span class="label label-info ">Set[String]</span> 
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--sample_file">--sample_file </a>
		 / <small>-sf</small>
	</h3>
	<p class="args">
		<b>File containing a list of samples to include</b><br />
		Sample names should be in a plain text file listing one sample name per line. This argument can be specified multiple times in order to provide
 multiple sample list files.
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
	<p>
		<span class="label label-info ">Set[File]</span> 
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--sample_name">--sample_name </a>
		 / <small>-sn</small>
	</h3>
	<p class="args">
		<b>Include genotypes from this sample</b><br />
		This argument can be specified multiple times in order to provide multiple sample names.
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
	<p>
		<span class="label label-info ">Set[String]</span> 
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--select_random_fraction">--select_random_fraction </a>
		 / <small>-fraction</small>
	</h3>
	<p class="args">
		<b>Select a fraction of variants at random from the input</b><br />
		The value of this argument should be a number between 0 and 1 specifying the fraction of total variants to be
 randomly selected from the input callset. Note that this is done using a probabilistic function, so the final
 result is not guaranteed to carry the exact fraction requested. Can be used for large fractions.
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
	<p>
		<span class="label label-info ">double</span> 
			&nbsp;<span class="label label-warning">[ [ -</span>
			&nbsp;<span class="label label-warning"> ] ]</span>
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--selectexpressions">--selectexpressions </a>
		 / <small>-select</small>
	</h3>
	<p class="args">
		<b>One or more criteria to use when selecting the data</b><br />
		See example commands above for detailed usage examples. Note that these expressions are evaluated *after* the
 specified samples are extracted and the INFO field annotations are updated.
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
	<p>
		<span class="label label-info ">ArrayList[String]</span> 
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--selectTypeToExclude">--selectTypeToExclude </a>
		 / <small>-xlSelectType</small>
	</h3>
	<p class="args">
		<b>Do not select certain type of variants from the input file</b><br />
		This argument excludes particular kinds of variants out of a list. If left empty, there is no type selection
 and all variant types are considered for other selection criteria. Valid types are INDEL, SNP, MIXED, MNP,
 SYMBOLIC, NO_VARIATION. Can be specified multiple times.
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
	<p>
		<span class="label label-info ">List[Type]</span> 
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--selectTypeToInclude">--selectTypeToInclude </a>
		 / <small>-selectType</small>
	</h3>
	<p class="args">
		<b>Select only a certain type of variants from the input file</b><br />
		This argument selects particular kinds of variants out of a list. If left empty, there is no type selection
 and all variant types are considered for other selection criteria. Valid types are INDEL, SNP, MIXED, MNP,
 SYMBOLIC, NO_VARIATION. Can be specified multiple times.
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
	<p>
		<span class="label label-info ">List[Type]</span> 
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--setFilteredGtToNocall">--setFilteredGtToNocall </a>
		 / <small>NA</small>
	</h3>
	<p class="args">
		<b>Set filtered genotypes to no-call</b><br />
		If this argument is provided, set filtered genotypes to no-call (./.).
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>NA</p> 
	-->
	<p>
		<span class="label label-info ">boolean</span> 
	</p>
	<hr style="border-bottom: dotted 1px #C0C0C0;" />
	<h3><a name="--variant">--variant </a>
		 / <small>-V</small>
	</h3>
	<p class="args">
		<b>Input VCF file</b><br />
		Variants from this VCF file are used by this tool as input.
 The file must at least contain the standard VCF header lines, but
 can be empty (i.e., no variants are contained in the file).
	</p>
	<!-- Not sure what this can contain apart from "required" and "deprecated" (which args are @hidden) so removing it provisionally -->
	<!--
		<p>required</p> 
	-->
		<p>--variant binds reference ordered data. This argument supports ROD files of the following types: <a href=htsjdk_variant_bcf2_BCF2Codec.html>BCF2</a>, <a href=htsjdk_variant_vcf_VCFCodec.html>VCF</a>, <a href=htsjdk_variant_vcf_VCF3Codec.html>VCF3</a></p>
	<p>
				<span class="badge badge-important">R</span>
		<span class="label label-info ">RodBinding[VariantContext]</span> 
	</p>
		




</div>

</div>  