

<div class='row' id="top">

	

	

	<section class="col-md-4">
		<aside class="well">
			<a href="index.html"><h4><i class='fa fa-chevron-left'></i> Back to Tool Docs Index</h4></a>
		</aside>
		<aside class="well">
			<h2>Categories</h2>

        <ul class="nav nav-pills nav-stacked" id="sidenav">
        		<hr>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#EngineParametersavailabletoalltools">Engine Parameters (available to all tools)</a>
							<div id="EngineParametersavailabletoalltools"
								
								<ul>
											<li>
												<a href="org_broadinstitute_gatk_engine_CommandLineGATK.html">CommandLineGATK</a>
											</li>
								</ul>
							</div>
						</li>
        		<hr>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#DiagnosticsandQualityControlTools">Diagnostics and Quality Control Tools</a>
							<div id="DiagnosticsandQualityControlTools"
								
								<ul>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_rnaseq_ASEReadCounter.html">ASEReadCounter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_bqsr_AnalyzeCovariates.html">AnalyzeCovariates</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_coverage_CallableLoci.html">CallableLoci</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_CheckPileup.html">CheckPileup</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_coverage_CompareCallableLoci.html">CompareCallableLoci</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_cancer_contamination_ContEst.html">ContEst</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_CountBases.html">CountBases</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_CountIntervals.html">CountIntervals</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_CountLoci.html">CountLoci</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_CountMales.html">CountMales</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_CountRODs.html">CountRODs</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_CountRODsByRef.html">CountRODsByRef</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_CountReadEvents.html">CountReadEvents</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_CountReads.html">CountReads</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_CountTerminusEvent.html">CountTerminusEvent</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_coverage_DepthOfCoverage.html">DepthOfCoverage</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_diagnostics_diagnosetargets_DiagnoseTargets.html">DiagnoseTargets</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_diffengine_DiffObjects.html">DiffObjects</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_diagnostics_ErrorRatePerCycle.html">ErrorRatePerCycle</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_fasta_FastaStats.html">FastaStats</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_diagnostics_FindCoveredIntervals.html">FindCoveredIntervals</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_FlagStat.html">FlagStat</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_coverage_GCContentByInterval.html">GCContentByInterval</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_Pileup.html">Pileup</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_PrintRODs.html">PrintRODs</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_diagnostics_missing_QualifyMissingIntervals.html">QualifyMissingIntervals</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_ReadClippingStats.html">ReadClippingStats</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_diagnostics_ReadGroupProperties.html">ReadGroupProperties</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_diagnostics_ReadLengthDistribution.html">ReadLengthDistribution</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_simulatereads_SimulateReadsForVariants.html">SimulateReadsForVariants</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#SequenceDataProcessingTools">Sequence Data Processing Tools</a>
							<div id="SequenceDataProcessingTools"
								
								<ul>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_bqsr_BaseRecalibrator.html">BaseRecalibrator</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_readutils_ClipReads.html">ClipReads</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_indels_IndelRealigner.html">IndelRealigner</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_indels_LeftAlignIndels.html">LeftAlignIndels</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_readutils_PrintReads.html">PrintReads</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_indels_RealignerTargetCreator.html">RealignerTargetCreator</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_rnaseq_SplitNCigarReads.html">SplitNCigarReads</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_readutils_SplitSamFile.html">SplitSamFile</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#VariantDiscoveryTools">Variant Discovery Tools</a>
							<div id="VariantDiscoveryTools"
								
								<ul>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantrecalibration_ApplyRecalibration.html">ApplyRecalibration</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_CalculateGenotypePosteriors.html">CalculateGenotypePosteriors</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_examples_GATKPaperGenotyper.html">GATKPaperGenotyper</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_GenotypeGVCFs.html">GenotypeGVCFs</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_haplotypecaller_HaplotypeCaller.html">HaplotypeCaller</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_cancer_m2_MuTect2.html">MuTect2</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_phasing_PhaseByTransmission.html">PhaseByTransmission</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_RegenotypeVariants.html">RegenotypeVariants</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_genotyper_UnifiedGenotyper.html">UnifiedGenotyper</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantrecalibration_VariantRecalibrator.html">VariantRecalibrator</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#VariantEvaluationTools">Variant Evaluation Tools</a>
							<div id="VariantEvaluationTools"
								
								<ul>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_GenotypeConcordance.html">GenotypeConcordance</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_ValidateVariants.html">ValidateVariants</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_varianteval_VariantEval.html">VariantEval</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_filters_VariantFiltration.html">VariantFiltration</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#VariantManipulationTools">Variant Manipulation Tools</a>
							<div id="VariantManipulationTools"
								
								<ul>
											<li>
												<a href="org_broadinstitute_gatk_tools_CatVariants.html">CatVariants</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_CombineGVCFs.html">CombineGVCFs</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_CombineVariants.html">CombineVariants</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_haplotypecaller_HaplotypeResolver.html">HaplotypeResolver</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_LeftAlignAndTrimVariants.html">LeftAlignAndTrimVariants</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_RandomlySplitVariants.html">RandomlySplitVariants</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_phasing_ReadBackedPhasing.html">ReadBackedPhasing</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_SelectHeaders.html">SelectHeaders</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_SelectVariants.html">SelectVariants</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_validation_validationsiteselector_ValidationSiteSelector.html">ValidationSiteSelector</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_VariantAnnotator.html">VariantAnnotator</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_VariantsToAllelicPrimitives.html">VariantsToAllelicPrimitives</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_VariantsToBinaryPed.html">VariantsToBinaryPed</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_VariantsToTable.html">VariantsToTable</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_VariantsToVCF.html">VariantsToVCF</a>
											</li>
								</ul>
							</div>
						</li>
        		<hr>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#AnnotationModules">Annotation Modules</a>
							<div id="AnnotationModules"
								
								<ul>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AS_BaseQualityRankSumTest.html">AS_BaseQualityRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AS_FisherStrand.html">AS_FisherStrand</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AS_InbreedingCoeff.html">AS_InbreedingCoeff</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AS_InsertSizeRankSum.html">AS_InsertSizeRankSum</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AS_MQMateRankSumTest.html">AS_MQMateRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AS_MappingQualityRankSumTest.html">AS_MappingQualityRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AS_QualByDepth.html">AS_QualByDepth</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AS_RMSMappingQuality.html">AS_RMSMappingQuality</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AS_ReadPosRankSumTest.html">AS_ReadPosRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AS_StrandOddsRatio.html">AS_StrandOddsRatio</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AlleleBalance.html">AlleleBalance</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AlleleBalanceBySample.html">AlleleBalanceBySample</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AlleleCountBySample.html">AlleleCountBySample</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_BaseCounts.html">BaseCounts</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_BaseCountsBySample.html">BaseCountsBySample</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_BaseQualityRankSumTest.html">BaseQualityRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_cancer_BaseQualitySumPerAlleleBySample.html">BaseQualitySumPerAlleleBySample</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_ChromosomeCounts.html">ChromosomeCounts</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_ClippingRankSumTest.html">ClippingRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_Coverage.html">Coverage</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_DepthPerAlleleBySample.html">DepthPerAlleleBySample</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_DepthPerSampleHC.html">DepthPerSampleHC</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_ExcessHet.html">ExcessHet</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_FisherStrand.html">FisherStrand</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_FractionInformativeReads.html">FractionInformativeReads</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_GCContent.html">GCContent</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_GenotypeSummaries.html">GenotypeSummaries</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_HaplotypeScore.html">HaplotypeScore</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_HardyWeinberg.html">HardyWeinberg</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_HomopolymerRun.html">HomopolymerRun</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_InbreedingCoeff.html">InbreedingCoeff</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_LikelihoodRankSumTest.html">LikelihoodRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_LowMQ.html">LowMQ</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_MVLikelihoodRatio.html">MVLikelihoodRatio</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_MappingQualityRankSumTest.html">MappingQualityRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_MappingQualityZero.html">MappingQualityZero</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_MappingQualityZeroBySample.html">MappingQualityZeroBySample</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_NBaseCount.html">NBaseCount</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_cancer_OxoGReadCounts.html">OxoGReadCounts</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_PossibleDeNovo.html">PossibleDeNovo</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_QualByDepth.html">QualByDepth</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_RMSMappingQuality.html">RMSMappingQuality</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_ReadPosRankSumTest.html">ReadPosRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_SampleList.html">SampleList</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_SnpEff.html">SnpEff</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_SpanningDeletions.html">SpanningDeletions</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_StrandAlleleCountsBySample.html">StrandAlleleCountsBySample</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_StrandBiasBySample.html">StrandBiasBySample</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_StrandOddsRatio.html">StrandOddsRatio</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_TandemRepeatAnnotator.html">TandemRepeatAnnotator</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_TransmissionDisequilibriumTest.html">TransmissionDisequilibriumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_VariantType.html">VariantType</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#ReadFilters">Read Filters</a>
							<div id="ReadFilters"
								
								<ul>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_BadCigarFilter.html">BadCigarFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_BadMateFilter.html">BadMateFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_CountingFilteringIterator$CountingReadFilter.html">CountingFilteringIterator.CountingReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_DuplicateReadFilter.html">DuplicateReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_FailsVendorQualityCheckFilter.html">FailsVendorQualityCheckFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_haplotypecaller_HCMappingQualityFilter.html">HCMappingQualityFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_LibraryReadFilter.html">LibraryReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_MalformedReadFilter.html">MalformedReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_MappingQualityFilter.html">MappingQualityFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_MappingQualityUnavailableFilter.html">MappingQualityUnavailableFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_MappingQualityZeroFilter.html">MappingQualityZeroFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_MateSameStrandFilter.html">MateSameStrandFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_MaxInsertSizeFilter.html">MaxInsertSizeFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_MissingReadGroupFilter.html">MissingReadGroupFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_NoOriginalQualityScoresFilter.html">NoOriginalQualityScoresFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_NotPrimaryAlignmentFilter.html">NotPrimaryAlignmentFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_OverclippedReadFilter.html">OverclippedReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_Platform454Filter.html">Platform454Filter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_PlatformFilter.html">PlatformFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_PlatformUnitFilter.html">PlatformUnitFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_ReadGroupBlackListFilter.html">ReadGroupBlackListFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_ReadLengthFilter.html">ReadLengthFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_ReadNameFilter.html">ReadNameFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_ReadStrandFilter.html">ReadStrandFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_ReassignMappingQualityFilter.html">ReassignMappingQualityFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_ReassignOneMappingQualityFilter.html">ReassignOneMappingQualityFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_ReassignOriginalMQAfterIndelRealignmentFilter.html">ReassignOriginalMQAfterIndelRealignmentFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_SampleFilter.html">SampleFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_SingleReadGroupFilter.html">SingleReadGroupFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_UnmappedReadFilter.html">UnmappedReadFilter</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#ResourceFileCodecs">Resource File Codecs</a>
							<div id="ResourceFileCodecs"
								
								<ul>
											<li>
												<a href="org_broadinstitute_gatk_utils_codecs_beagle_BeagleCodec.html">BeagleCodec</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_utils_codecs_table_BedTableCodec.html">BedTableCodec</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_utils_codecs_hapmap_RawHapMapCodec.html">RawHapMapCodec</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_utils_codecs_refseq_RefSeqCodec.html">RefSeqCodec</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_utils_codecs_sampileup_SAMPileupCodec.html">SAMPileupCodec</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_utils_codecs_samread_SAMReadCodec.html">SAMReadCodec</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_utils_codecs_table_TableCodec.html">TableCodec</a>
											</li>
								</ul>
							</div>
						</li>
        		<hr>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#ReferenceUtilities">Reference Utilities</a>
							<div id="ReferenceUtilities"
								
								<ul>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_fasta_FastaAlternateReferenceMaker.html">FastaAlternateReferenceMaker</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_fasta_FastaReferenceMaker.html">FastaReferenceMaker</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_QCRef.html">QCRef</a>
											</li>
								</ul>
							</div>
						</li>
        </ul>
		</aside>
		

	</section>

	<div class="col-md-8">           <h1><small>Showing docs for version 3.6-0 </small></h1>     <hr />

		<h1>MuTect2</h1>

		<p class="lead">Call somatic SNPs and indels via local re-assembly of haplotypes</p>

			<h3>Category
				<small> Variant Discovery Tools</small>
			</h3>
			<h3>Traversal
				<small>ActiveRegionWalker</small>
			</h3>
			<h3>PartitionBy
				<small>LOCUS</small>
			</h3>

		<hr>
		<h2>Overview</h2>
		

 <p>MuTect2 is a somatic SNP and indel caller that combines the DREAM challenge-winning somatic genotyping engine of the original MuTect (<a href='http://www.nature.com/nbt/journal/v31/n3/full/nbt.2514.html'>Cibulskis et al., 2013</a>) with the assembly-based machinery of HaplotypeCaller.</p>

 <p>The basic operation of MuTect2 proceeds similarly to that of the <a href="https://www.broadinstitute.org/gatk/guide/tooldocs/org_broadinstitute_gatk_tools_walkers_haplotypecaller_HaplotypeCaller.html">HaplotypeCaller</a>   </p>

 <h3>Differences from HaplotypeCaller</h3>
 <p>While the HaplotypeCaller relies on a ploidy assumption (diploid by default) to inform its genotype likelihood and
 variant quality calculations, MuTect2 allows for a varying allelic fraction for each variant, as is often seen in tumors with purity less
 than 100%, multiple subclones, and/or copy number variation (either local or aneuploidy). MuTect2 also differs from the HaplotypeCaller in that it does apply some hard filters
 to variants before producing output.</p>

 <h3>Usage examples</h3>
 <p>These are example commands that show how to run MuTect2 for typical use cases. Square brackets ("[ ]")
 indicate optional arguments. Note that parameter values shown here may not be the latest recommended; see the
 Best Practices documentation for detailed recommendations. </p>

 <br />
 <h4>Tumor/Normal variant calling</h4>
 <pre>
   java
     -jar GenomeAnalysisTK.jar \
     -T MuTect2 \
     -R reference.fasta \
     -I:tumor tumor.bam \
     -I:normal normal.bam \
     [--dbsnp dbSNP.vcf] \
     [--cosmic COSMIC.vcf] \
     [-L targets.interval_list] \
     -o output.vcf
 </pre>

 <h4>Normal-only calling for panel of normals creation</h4>
 <pre>
   java
     -jar GenomeAnalysisTK.jar
     -T MuTect2
     -R reference.fasta
     -I:tumor normal1.bam \
     [--dbsnp dbSNP.vcf] \
     [--cosmic COSMIC.vcf] \
     --artifact_detection_mode \
     [-L targets.interval_list] \
     -o output.normal1.vcf
 </pre>
 <br />
 For full PON creation, call each of your normals separately in artifact detection mode. Then use CombineVariants to
 output only sites where a variant was seen in at least two samples:
 <pre>
 java -jar GenomeAnalysisTK.jar
     -T CombineVariants
     -R reference.fasta
     -V output.normal1.vcf -V output.normal2.vcf [-V output.normal2.vcf ...] \
     -minN 2 \
     --setKey "null" \
     --filteredAreUncalled \
     --filteredrecordsmergetype KEEP_IF_ANY_UNFILTERED \
     [-L targets.interval_list] \
     -o MuTect2_PON.vcf
 </pre>

 <h3>Caveats</h3>
 <ul>
 <li>MuTect2 currently only supports the calling of a single tumor-normal pair at a time</li>
 </ul>

				<hr>
				<h2>Additional Information</h2>
				<p></p>
				<h3>Read filters</h3>
					<p>These Read Filters are automatically applied to the data by the Engine before processing by MuTect2.</p>
				<ul>
						<li><a href="org_broadinstitute_gatk_engine_filters_MalformedReadFilter.html">MalformedReadFilter</a></li>
						<li><a href="org_broadinstitute_gatk_engine_filters_BadCigarFilter.html">BadCigarFilter</a></li>
						<li><a href="org_broadinstitute_gatk_engine_filters_UnmappedReadFilter.html">UnmappedReadFilter</a></li>
						<li><a href="org_broadinstitute_gatk_engine_filters_NotPrimaryAlignmentFilter.html">NotPrimaryAlignmentFilter</a></li>
						<li><a href="org_broadinstitute_gatk_engine_filters_FailsVendorQualityCheckFilter.html">FailsVendorQualityCheckFilter</a></li>
						<li><a href="org_broadinstitute_gatk_engine_filters_DuplicateReadFilter.html">DuplicateReadFilter</a></li>
						<li><a href="org_broadinstitute_gatk_engine_filters_MappingQualityUnavailableFilter.html">MappingQualityUnavailableFilter</a></li>
				</ul>
				<h3>Parallelism options</h3>
					<p>This tool can be run in multi-threaded mode using this option.</p>
				<ul>
						<li><a href="https://www.broadinstitute.org/gatk/guide/tooldocs/org_broadinstitute_gatk_engine_CommandLineGATK.html#-nct">NanoSchedulable (-nct)</a></li>
				</ul>
				<h3>Downsampling settings</h3>
					<p>This tool applies the following downsampling settings by default.</p>
					<ul>
						<li>Mode: <b>BY_SAMPLE</b></li>
						<li>To coverage: <b>1,000</b></li>
					</ul>
				<h3>ActiveRegion settings</h3>
				<p>This tool uses ActiveRegions on the reference.</p>
				<ul>
					<li>Minimum region size: <b>50</b> bp</li>
					<li>Maximum region size: <b>300</b> bp</li>
					<li>Extension increments: <b>100</b> bp</li>
				</ul>
				<hr>
				<h2>Command-line Arguments</h2>
				<p></p>
				<h3>Engine arguments</h3>
				<p>All tools inherit arguments from the GATK Engine' "CommandLineGATK" argument collection, which can be
					used to modify various aspects of the tool's function. For example, the -L argument directs the GATK
					engine to restrict processing to specific genomic intervals; or the -rf argument allows you to apply
					certain read filters to exclude some of the data from the analysis.</p>
				<ul>
						<li><a href="org_broadinstitute_gatk_engine_CommandLineGATK.html">CommandLineGATK</a></li>
				</ul>


					<h3>MuTect2 specific arguments</h3>
					<p>This table summarizes the command-line arguments that are specific to this tool. For more details on each argument, see the list further down below the table or click on an argument name to jump directly to that entry in the list.</p>
					<table class="table table-striped table-bordered table-condensed">
						<thead>
						<tr>
							<th>Argument name(s)</th>
							<th>Default value</th>
							<th>Summary</th>
						</tr>
						</thead>
						<tbody>
			<tr>
				<th colspan="4" id="row-divider">Optional Inputs</th>
			</tr>
				<tr>
					<td><a href="#--alleles">--alleles</a><br />
					</td>
					<!--<td>RodBinding[VariantContext]</td> -->
					<td>none</td>
					<td>The set of alleles at which to genotype when --genotyping_mode is GENOTYPE_GIVEN_ALLELES</td>
				</tr>
				<tr>
					<td><a href="#--cosmic">--cosmic</a><br />
					</td>
					<!--<td>List[RodBinding[VariantContext]]</td> -->
					<td>[]</td>
					<td>VCF file of COSMIC sites</td>
				</tr>
				<tr>
					<td><a href="#--dbsnp">--dbsnp</a><br />
								&nbsp;<em>-D</em>
					</td>
					<!--<td>RodBinding[VariantContext]</td> -->
					<td>none</td>
					<td>dbSNP file</td>
				</tr>
				<tr>
					<td><a href="#--normal_panel">--normal_panel</a><br />
								&nbsp;<em>-PON</em>
					</td>
					<!--<td>List[RodBinding[VariantContext]]</td> -->
					<td>[]</td>
					<td>VCF file of sites observed in normal</td>
				</tr>
			<tr>
				<th colspan="4" id="row-divider">Optional Outputs</th>
			</tr>
				<tr>
					<td><a href="#--activeRegionOut">--activeRegionOut</a><br />
								&nbsp;<em>-ARO</em>
					</td>
					<!--<td>PrintStream</td> -->
					<td>NA</td>
					<td>Output the active region to this IGV formatted file</td>
				</tr>
				<tr>
					<td><a href="#--activityProfileOut">--activityProfileOut</a><br />
								&nbsp;<em>-APO</em>
					</td>
					<!--<td>PrintStream</td> -->
					<td>NA</td>
					<td>Output the raw activity profile results in IGV format</td>
				</tr>
				<tr>
					<td><a href="#--graphOutput">--graphOutput</a><br />
								&nbsp;<em>-graph</em>
					</td>
					<!--<td>PrintStream</td> -->
					<td>NA</td>
					<td>Write debug assembly graph information to this file</td>
				</tr>
				<tr>
					<td><a href="#--out">--out</a><br />
								&nbsp;<em>-o</em>
					</td>
					<!--<td>VariantContextWriter</td> -->
					<td>stdout</td>
					<td>File to which variants should be written</td>
				</tr>
			<tr>
				<th colspan="4" id="row-divider">Optional Parameters</th>
			</tr>
				<tr>
					<td><a href="#--contamination_fraction_to_filter">--contamination_fraction_to_filter</a><br />
								&nbsp;<em>-contamination</em>
					</td>
					<!--<td>double</td> -->
					<td>0.0</td>
					<td>Fraction of contamination in sequencing data (for all samples) to aggressively remove</td>
				</tr>
				<tr>
					<td><a href="#--dbsnp_normal_lod">--dbsnp_normal_lod</a><br />
					</td>
					<!--<td>double</td> -->
					<td>5.5</td>
					<td>LOD threshold for calling normal non-variant at dbsnp sites</td>
				</tr>
				<tr>
					<td><a href="#--debug_read_name">--debug_read_name</a><br />
					</td>
					<!--<td>String</td> -->
					<td>NA</td>
					<td>trace this read name through the calling process</td>
				</tr>
				<tr>
					<td><a href="#--genotyping_mode">--genotyping_mode</a><br />
								&nbsp;<em>-gt_mode</em>
					</td>
					<!--<td>GenotypingOutputMode</td> -->
					<td>DISCOVERY</td>
					<td>Specifies how to determine the alternate alleles to use for genotyping</td>
				</tr>
				<tr>
					<td><a href="#--group">--group</a><br />
								&nbsp;<em>-G</em>
					</td>
					<!--<td>String[]</td> -->
					<td>[]</td>
					<td>One or more classes/groups of annotations to apply to variant calls</td>
				</tr>
				<tr>
					<td><a href="#--heterozygosity">--heterozygosity</a><br />
								&nbsp;<em>-hets</em>
					</td>
					<!--<td>Double</td> -->
					<td>0.001</td>
					<td>Heterozygosity value used to compute prior likelihoods for any locus</td>
				</tr>
				<tr>
					<td><a href="#--indel_heterozygosity">--indel_heterozygosity</a><br />
								&nbsp;<em>-indelHeterozygosity</em>
					</td>
					<!--<td>double</td> -->
					<td>1.25E-4</td>
					<td>Heterozygosity for indel calling</td>
				</tr>
				<tr>
					<td><a href="#--initial_normal_lod">--initial_normal_lod</a><br />
					</td>
					<!--<td>double</td> -->
					<td>0.5</td>
					<td>Initial LOD threshold for calling normal variant</td>
				</tr>
				<tr>
					<td><a href="#--initial_tumor_lod">--initial_tumor_lod</a><br />
					</td>
					<!--<td>double</td> -->
					<td>4.0</td>
					<td>Initial LOD threshold for calling tumor variant</td>
				</tr>
				<tr>
					<td><a href="#--max_alt_allele_in_normal_fraction">--max_alt_allele_in_normal_fraction</a><br />
					</td>
					<!--<td>double</td> -->
					<td>0.03</td>
					<td>Threshold for maximum alternate allele fraction in normal</td>
				</tr>
				<tr>
					<td><a href="#--max_alt_alleles_in_normal_count">--max_alt_alleles_in_normal_count</a><br />
					</td>
					<!--<td>int</td> -->
					<td>1</td>
					<td>Threshold for maximum alternate allele counts in normal</td>
				</tr>
				<tr>
					<td><a href="#--max_alt_alleles_in_normal_qscore_sum">--max_alt_alleles_in_normal_qscore_sum</a><br />
					</td>
					<!--<td>int</td> -->
					<td>20</td>
					<td>Threshold for maximum alternate allele quality score sum in normal</td>
				</tr>
				<tr>
					<td><a href="#--min_base_quality_score">--min_base_quality_score</a><br />
								&nbsp;<em>-mbq</em>
					</td>
					<!--<td>byte</td> -->
					<td>10</td>
					<td>Minimum base quality required to consider a base for calling</td>
				</tr>
				<tr>
					<td><a href="#--normal_lod">--normal_lod</a><br />
					</td>
					<!--<td>double</td> -->
					<td>2.2</td>
					<td>LOD threshold for calling normal non-germline</td>
				</tr>
				<tr>
					<td><a href="#--output_mode">--output_mode</a><br />
								&nbsp;<em>-out_mode</em>
					</td>
					<!--<td>OutputMode</td> -->
					<td>EMIT_VARIANTS_ONLY</td>
					<td>Specifies which type of calls we should output</td>
				</tr>
				<tr>
					<td><a href="#--sample_ploidy">--sample_ploidy</a><br />
								&nbsp;<em>-ploidy</em>
					</td>
					<!--<td>int</td> -->
					<td>2</td>
					<td>Ploidy (number of chromosomes) per sample. For pooled data, set to (Number of samples in each pool * Sample Ploidy).</td>
				</tr>
				<tr>
					<td><a href="#--standard_min_confidence_threshold_for_calling">--standard_min_confidence_threshold_for_calling</a><br />
								&nbsp;<em>-stand_call_conf</em>
					</td>
					<!--<td>double</td> -->
					<td>30.0</td>
					<td>The minimum phred-scaled confidence threshold at which variants should be called</td>
				</tr>
				<tr>
					<td><a href="#--standard_min_confidence_threshold_for_emitting">--standard_min_confidence_threshold_for_emitting</a><br />
								&nbsp;<em>-stand_emit_conf</em>
					</td>
					<!--<td>double</td> -->
					<td>30.0</td>
					<td>The minimum phred-scaled confidence threshold at which variants should be emitted (and filtered with LowQual if less than the calling threshold)</td>
				</tr>
				<tr>
					<td><a href="#--tumor_lod">--tumor_lod</a><br />
					</td>
					<!--<td>double</td> -->
					<td>6.3</td>
					<td>LOD threshold for calling tumor variant</td>
				</tr>
			<tr>
				<th colspan="4" id="row-divider">Optional Flags</th>
			</tr>
				<tr>
					<td><a href="#--annotateNDA">--annotateNDA</a><br />
								&nbsp;<em>-nda</em>
					</td>
					<!--<td>boolean</td> -->
					<td>false</td>
					<td>If provided, we will annotate records with the number of alternate alleles that were discovered (but not necessarily genotyped) at a given site</td>
				</tr>
			<tr>
				<th colspan="4" id="row-divider">Advanced Inputs</th>
			</tr>
				<tr>
					<td><a href="#--activeRegionIn">--activeRegionIn</a><br />
								&nbsp;<em>-AR</em>
					</td>
					<!--<td>List[IntervalBinding[Feature]]</td> -->
					<td>NA</td>
					<td>Use this interval list file as the active regions to process</td>
				</tr>
				<tr>
					<td><a href="#--comp">--comp</a><br />
					</td>
					<!--<td>List[RodBinding[VariantContext]]</td> -->
					<td>[]</td>
					<td>comparison VCF file</td>
				</tr>
			<tr>
				<th colspan="4" id="row-divider">Advanced Outputs</th>
			</tr>
				<tr>
					<td><a href="#--bamOutput">--bamOutput</a><br />
								&nbsp;<em>-bamout</em>
					</td>
					<!--<td>GATKSAMFileWriter</td> -->
					<td>NA</td>
					<td>File to which assembled haplotypes should be written</td>
				</tr>
			<tr>
				<th colspan="4" id="row-divider">Advanced Parameters</th>
			</tr>
				<tr>
					<td><a href="#--activeProbabilityThreshold">--activeProbabilityThreshold</a><br />
								&nbsp;<em>-ActProbThresh</em>
					</td>
					<!--<td>Double</td> -->
					<td>0.002</td>
					<td>Threshold for the probability of a profile state being active.</td>
				</tr>
				<tr>
					<td><a href="#--activeRegionExtension">--activeRegionExtension</a><br />
					</td>
					<!--<td>Integer</td> -->
					<td>NA</td>
					<td>The active region extension; if not provided defaults to Walker annotated default</td>
				</tr>
				<tr>
					<td><a href="#--activeRegionMaxSize">--activeRegionMaxSize</a><br />
					</td>
					<!--<td>Integer</td> -->
					<td>NA</td>
					<td>The active region maximum size; if not provided defaults to Walker annotated default</td>
				</tr>
				<tr>
					<td><a href="#--annotation">--annotation</a><br />
								&nbsp;<em>-A</em>
					</td>
					<!--<td>List[String]</td> -->
					<td>[DepthPerAlleleBySample, BaseQualitySumPerAlleleBySample, TandemRepeatAnnotator, OxoGReadCounts]</td>
					<td>One or more specific annotations to apply to variant calls</td>
				</tr>
				<tr>
					<td><a href="#--bamWriterType">--bamWriterType</a><br />
					</td>
					<!--<td>Type</td> -->
					<td>CALLED_HAPLOTYPES</td>
					<td>Which haplotypes should be written to the BAM</td>
				</tr>
				<tr>
					<td><a href="#--bandPassSigma">--bandPassSigma</a><br />
					</td>
					<!--<td>Double</td> -->
					<td>NA</td>
					<td>The sigma of the band pass filter Gaussian kernel; if not provided defaults to Walker annotated default</td>
				</tr>
				<tr>
					<td><a href="#--contamination_fraction_per_sample_file">--contamination_fraction_per_sample_file</a><br />
								&nbsp;<em>-contaminationFile</em>
					</td>
					<!--<td>File</td> -->
					<td>NA</td>
					<td>Tab-separated File containing fraction of contamination in sequencing data (per sample) to aggressively remove. Format should be "<SampleID><TAB><Contamination>" (Contamination is double) per line; No header.</td>
				</tr>
				<tr>
					<td><a href="#--emitRefConfidence">--emitRefConfidence</a><br />
								&nbsp;<em>-ERC</em>
					</td>
					<!--<td>ReferenceConfidenceMode</td> -->
					<td>NONE</td>
					<td>Mode for emitting reference confidence scores</td>
				</tr>
				<tr>
					<td><a href="#--excludeAnnotation">--excludeAnnotation</a><br />
								&nbsp;<em>-XA</em>
					</td>
					<!--<td>List[String]</td> -->
					<td>[SpanningDeletions]</td>
					<td>One or more specific annotations to exclude</td>
				</tr>
				<tr>
					<td><a href="#--gcpHMM">--gcpHMM</a><br />
					</td>
					<!--<td>int</td> -->
					<td>10</td>
					<td>Flat gap continuation penalty for use in the Pair HMM</td>
				</tr>
				<tr>
					<td><a href="#--input_prior">--input_prior</a><br />
								&nbsp;<em>-inputPrior</em>
					</td>
					<!--<td>List[Double]</td> -->
					<td>[]</td>
					<td>Input prior for calls</td>
				</tr>
				<tr>
					<td><a href="#--kmerSize">--kmerSize</a><br />
					</td>
					<!--<td>List[Integer]</td> -->
					<td>[10, 25]</td>
					<td>Kmer size to use in the read threading assembler</td>
				</tr>
				<tr>
					<td><a href="#--max_alternate_alleles">--max_alternate_alleles</a><br />
								&nbsp;<em>-maxAltAlleles</em>
					</td>
					<!--<td>int</td> -->
					<td>6</td>
					<td>Maximum number of alternate alleles to genotype</td>
				</tr>
				<tr>
					<td><a href="#--max_num_PL_values">--max_num_PL_values</a><br />
								&nbsp;<em>-maxNumPLValues</em>
					</td>
					<!--<td>int</td> -->
					<td>100</td>
					<td>Maximum number of PL values to output</td>
				</tr>
				<tr>
					<td><a href="#--maxNumHaplotypesInPopulation">--maxNumHaplotypesInPopulation</a><br />
					</td>
					<!--<td>int</td> -->
					<td>128</td>
					<td>Maximum number of haplotypes to consider for your population</td>
				</tr>
				<tr>
					<td><a href="#--minDanglingBranchLength">--minDanglingBranchLength</a><br />
					</td>
					<!--<td>int</td> -->
					<td>4</td>
					<td>Minimum length of a dangling branch to attempt recovery</td>
				</tr>
				<tr>
					<td><a href="#--minPruning">--minPruning</a><br />
					</td>
					<!--<td>int</td> -->
					<td>2</td>
					<td>Minimum support to not prune paths in the graph</td>
				</tr>
				<tr>
					<td><a href="#--numPruningSamples">--numPruningSamples</a><br />
					</td>
					<!--<td>int</td> -->
					<td>1</td>
					<td>Number of samples that must pass the minPruning threshold</td>
				</tr>
				<tr>
					<td><a href="#--phredScaledGlobalReadMismappingRate">--phredScaledGlobalReadMismappingRate</a><br />
								&nbsp;<em>-globalMAPQ</em>
					</td>
					<!--<td>int</td> -->
					<td>45</td>
					<td>The global assumed mismapping rate for reads</td>
				</tr>
			<tr>
				<th colspan="4" id="row-divider">Advanced Flags</th>
			</tr>
				<tr>
					<td><a href="#--allowNonUniqueKmersInRef">--allowNonUniqueKmersInRef</a><br />
					</td>
					<!--<td>boolean</td> -->
					<td>false</td>
					<td>Allow graphs that have non-unique kmers in the reference</td>
				</tr>
				<tr>
					<td><a href="#--allSitePLs">--allSitePLs</a><br />
					</td>
					<!--<td>boolean</td> -->
					<td>false</td>
					<td>Annotate all sites with PLs</td>
				</tr>
				<tr>
					<td><a href="#--artifact_detection_mode">--artifact_detection_mode</a><br />
					</td>
					<!--<td>boolean</td> -->
					<td>false</td>
					<td>Enable artifact detection for creating panels of normals</td>
				</tr>
				<tr>
					<td><a href="#--consensus">--consensus</a><br />
					</td>
					<!--<td>boolean</td> -->
					<td>false</td>
					<td>1000G consensus mode</td>
				</tr>
				<tr>
					<td><a href="#--debug">--debug</a><br />
					</td>
					<!--<td>boolean</td> -->
					<td>false</td>
					<td>Print out very verbose debug information about each triggering active region</td>
				</tr>
				<tr>
					<td><a href="#--disableOptimizations">--disableOptimizations</a><br />
					</td>
					<!--<td>boolean</td> -->
					<td>false</td>
					<td>Don't skip calculations in ActiveRegions with no variants</td>
				</tr>
				<tr>
					<td><a href="#--doNotRunPhysicalPhasing">--doNotRunPhysicalPhasing</a><br />
					</td>
					<!--<td>boolean</td> -->
					<td>false</td>
					<td>Disable physical phasing</td>
				</tr>
				<tr>
					<td><a href="#--dontIncreaseKmerSizesForCycles">--dontIncreaseKmerSizesForCycles</a><br />
					</td>
					<!--<td>boolean</td> -->
					<td>false</td>
					<td>Disable iterating over kmer sizes when graph cycles are detected</td>
				</tr>
				<tr>
					<td><a href="#--dontTrimActiveRegions">--dontTrimActiveRegions</a><br />
					</td>
					<!--<td>boolean</td> -->
					<td>false</td>
					<td>If specified, we will not trim down the active region from the full region (active + extension) to just the active interval for genotyping</td>
				</tr>
				<tr>
					<td><a href="#--dontUseSoftClippedBases">--dontUseSoftClippedBases</a><br />
					</td>
					<!--<td>boolean</td> -->
					<td>false</td>
					<td>If specified, we will not analyze soft clipped bases in the reads</td>
				</tr>
				<tr>
					<td><a href="#--emitDroppedReads">--emitDroppedReads</a><br />
								&nbsp;<em>-edr</em>
					</td>
					<!--<td>boolean</td> -->
					<td>false</td>
					<td>Emit reads that are dropped for filtering, trimming, realignment failure</td>
				</tr>
				<tr>
					<td><a href="#--forceActive">--forceActive</a><br />
					</td>
					<!--<td>boolean</td> -->
					<td>false</td>
					<td>If provided, all bases will be tagged as active</td>
				</tr>
				<tr>
					<td><a href="#--m2debug">--m2debug</a><br />
					</td>
					<!--<td>boolean</td> -->
					<td>false</td>
					<td>Print out very verbose M2 debug information</td>
				</tr>
				<tr>
					<td><a href="#--useFilteredReadsForAnnotations">--useFilteredReadsForAnnotations</a><br />
					</td>
					<!--<td>boolean</td> -->
					<td>false</td>
					<td>Use the contamination-filtered read maps for the purposes of annotating variants</td>
				</tr>
						</tbody>
					</table>

							<h3>Argument details</h3>
							<p>Arguments in this list are specific to this tool. Keep in mind that other arguments are available that are shared with other tools (e.g. command-line GATK arguments); see Inherited arguments above.</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--activeProbabilityThreshold">--activeProbabilityThreshold </a>
			 / <small>-ActProbThresh</small>
		</h3>
		<p class="args">
			<b>Threshold for the probability of a profile state being active.</b><br />
			
		</p>
		<p>
			<span class="label label-info ">Double</span>
				&nbsp;<span class="label">0.002</span>
				&nbsp;<span class="label label-warning">[ [ 0</span>
				&nbsp;<span class="label label-warning">1 ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--activeRegionExtension">--activeRegionExtension </a>
			 / <small>-activeRegionExtension</small>
		</h3>
		<p class="args">
			<b>The active region extension; if not provided defaults to Walker annotated default</b><br />
			
		</p>
		<p>
			<span class="label label-info ">Integer</span>
				&nbsp;<span class="label">NA</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--activeRegionIn">--activeRegionIn </a>
			 / <small>-AR</small>
		</h3>
		<p class="args">
			<b>Use this interval list file as the active regions to process</b><br />
			
		</p>
		<p>
			<span class="label label-info ">List[IntervalBinding[Feature]]</span>
				&nbsp;<span class="label">NA</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--activeRegionMaxSize">--activeRegionMaxSize </a>
			 / <small>-activeRegionMaxSize</small>
		</h3>
		<p class="args">
			<b>The active region maximum size; if not provided defaults to Walker annotated default</b><br />
			
		</p>
		<p>
			<span class="label label-info ">Integer</span>
				&nbsp;<span class="label">NA</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--activeRegionOut">--activeRegionOut </a>
			 / <small>-ARO</small>
		</h3>
		<p class="args">
			<b>Output the active region to this IGV formatted file</b><br />
			If provided, this walker will write out its active and inactive regions
 to this file in the IGV formatted TAB deliminated output:

 http://www.broadinstitute.org/software/igv/IGV

 Intended to make debugging the active region calculations easier
		</p>
		<p>
			<span class="label label-info ">PrintStream</span>
				&nbsp;<span class="label">NA</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--activityProfileOut">--activityProfileOut </a>
			 / <small>-APO</small>
		</h3>
		<p class="args">
			<b>Output the raw activity profile results in IGV format</b><br />
			If provided, this walker will write out its activity profile (per bp probabilities of being active)
 to this file in the IGV formatted TAB deliminated output:

 http://www.broadinstitute.org/software/igv/IGV

 Intended to make debugging the activity profile calculations easier
		</p>
		<p>
			<span class="label label-info ">PrintStream</span>
				&nbsp;<span class="label">NA</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--alleles">--alleles </a>
			 / <small>-alleles</small>
		</h3>
		<p class="args">
			<b>The set of alleles at which to genotype when --genotyping_mode is GENOTYPE_GIVEN_ALLELES</b><br />
			When the UnifiedGenotyper is put into GENOTYPE_GIVEN_ALLELES mode it will genotype the samples using only the alleles provide in this rod binding
		</p>
			<p>This argument supports reference-ordered data (ROD) files in the following formats: BCF2, VCF, VCF3</p>
		<p>
			<span class="label label-info ">RodBinding[VariantContext]</span>
				&nbsp;<span class="label">none</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--allowNonUniqueKmersInRef">--allowNonUniqueKmersInRef </a>
			 / <small>-allowNonUniqueKmersInRef</small>
		</h3>
		<p class="args">
			<b>Allow graphs that have non-unique kmers in the reference</b><br />
			By default, the program does not allow processing of reference sections that contain non-unique kmers. Disabling
 this check may cause problems in the assembly graph.
		</p>
		<p>
			<span class="label label-info ">boolean</span>
				&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--allSitePLs">--allSitePLs </a>
			 / <small>-allSitePLs</small>
		</h3>
		<p class="args">
			<b>Annotate all sites with PLs</b><br />
			Advanced, experimental argument: if SNP likelihood model is specified, and if EMIT_ALL_SITES output mode is set, when we set this argument then we will also emit PLs at all sites.
 This will give a measure of reference confidence and a measure of which alt alleles are more plausible (if any).
 WARNINGS:
 - This feature will inflate VCF file size considerably.
 - All SNP ALT alleles will be emitted with corresponding 10 PL values.
 - An error will be emitted if EMIT_ALL_SITES is not set, or if anything other than diploid SNP model is used
		</p>
		<p>
			<span class="label label-info ">boolean</span>
				&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--annotateNDA">--annotateNDA </a>
			 / <small>-nda</small>
		</h3>
		<p class="args">
			<b>If provided, we will annotate records with the number of alternate alleles that were discovered (but not necessarily genotyped) at a given site</b><br />
			Depending on the value of the --max_alternate_alleles argument, we may genotype only a fraction of the alleles being sent on for genotyping.
 Using this argument instructs the genotyper to annotate (in the INFO field) the number of alternate alleles that were originally discovered at the site.
		</p>
		<p>
			<span class="label label-info ">boolean</span>
				&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--annotation">--annotation </a>
			 / <small>-A</small>
		</h3>
		<p class="args">
			<b>One or more specific annotations to apply to variant calls</b><br />
			Which annotations to add to the output VCF file. See the VariantAnnotator -list argument to view available annotations.
		</p>
		<p>
			<span class="label label-info ">List[String]</span>
				&nbsp;<span class="label">[DepthPerAlleleBySample, BaseQualitySumPerAlleleBySample, TandemRepeatAnnotator, OxoGReadCounts]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--artifact_detection_mode">--artifact_detection_mode </a>
			 / <small>NA</small>
		</h3>
		<p class="args">
			<b>Enable artifact detection for creating panels of normals</b><br />
			Artifact detection mode is used to prepare a panel of normals. This maintains the specified tumor LOD threshold,
 but disables the remaining pragmatic filters. See usage examples above for more information.
		</p>
		<p>
			<span class="label label-info ">boolean</span>
				&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--bamOutput">--bamOutput </a>
			 / <small>-bamout</small>
		</h3>
		<p class="args">
			<b>File to which assembled haplotypes should be written</b><br />
			The assembled haplotypes and locally realigned reads will be written as BAM to this file if requested.  This is
 intended to be used only for troubleshooting purposes, in specific areas where you want to better understand
 why the caller is making specific calls. Turning on this mode may result in serious performance cost for the
 caller, so we do NOT recommend using this argument systematically as it will significantly increase runtime.

 The candidate haplotypes (called or all, depending on mode) are emitted as single reads covering the entire
 active region, coming from sample "HC" and a special read group called "ArtificialHaplotype". This will increase
 the pileup depth compared to what would be expected from the reads only, especially in complex regions.

 The reads are written out containing an "HC" tag (integer) that encodes which haplotype each read best matches
 according to the haplotype caller's likelihood calculation.  The use of this tag is primarily intended
 to allow good coloring of reads in IGV.  Simply go to "Color Alignments By > Tag" and enter "HC" to more
 easily see which reads go with these haplotype. You can also tell IGV to group reads by sample, which will
 separate the potential haplotypes from the reads. These features are illustrated in
 <a href="https://www.dropbox.com/s/xvy7sbxpf13x5bp/haplotypecaller%20bamout%20for%20docs.png">this screenshot</a>.

 Note that only reads that are actually informative about the haplotypes are emitted with the HC tag.
 By informative we mean that there's a meaningful difference in the likelihood of the read coming from one
 haplotype compared to the next best haplotype. When coloring reads by HC tag in IGV, uninformative reads will
 remain grey.

 Note also that not every input read is emitted to the bam in this mode. To include all trimmed, downsampled,
 filtered and uninformative reads, add the <code>--emitDroppedReads</code> argument.

 If multiple BAMs are passed as input to the tool (as is common for MuTect2), then they will be combined in the
 <code>-bamout</code> output and tagged with the appropriate sample names.
		</p>
		<p>
			<span class="label label-info ">GATKSAMFileWriter</span>
				&nbsp;<span class="label">NA</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--bamWriterType">--bamWriterType </a>
			 / <small>-bamWriterType</small>
		</h3>
		<p class="args">
			<b>Which haplotypes should be written to the BAM</b><br />
			The type of <code>-bamout</code> output we want to see. This determines whether HC will write out all of the haplotypes it
 considered (top 128 max) or just the ones that were selected as alleles and assigned to samples.
		</p>
			<p>
			The --bamWriterType argument is an enumerated type (Type), which can have one of the following values:
			<dl class="enum">
					<dt class="enum">ALL_POSSIBLE_HAPLOTYPES</dt>
					<dd class="enum">A mode that's for method developers.  Writes out all of the possible
 haplotypes considered, as well as reads aligned to each</dd>
					<dt class="enum">CALLED_HAPLOTYPES</dt>
					<dd class="enum">A mode for users.  Writes out the reads aligned only to the called
 haplotypes.  Useful to understand why the caller is calling what it is</dd>
			</dl>
			</p>
		<p>
			<span class="label label-info ">Type</span>
				&nbsp;<span class="label">CALLED_HAPLOTYPES</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--bandPassSigma">--bandPassSigma </a>
			 / <small>-bandPassSigma</small>
		</h3>
		<p class="args">
			<b>The sigma of the band pass filter Gaussian kernel; if not provided defaults to Walker annotated default</b><br />
			
		</p>
		<p>
			<span class="label label-info ">Double</span>
				&nbsp;<span class="label">NA</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--comp">--comp </a>
			 / <small>-comp</small>
		</h3>
		<p class="args">
			<b>comparison VCF file</b><br />
			If a call overlaps with a record from the provided comp track, the INFO field will be annotated
  as such in the output with the track name (e.g. -comp:FOO will have 'FOO' in the INFO field).
  Records that are filtered in the comp track will be ignored.
  Note that 'dbSNP' has been special-cased (see the --dbsnp argument).
		</p>
			<p>This argument supports reference-ordered data (ROD) files in the following formats: BCF2, VCF, VCF3</p>
		<p>
			<span class="label label-info ">List[RodBinding[VariantContext]]</span>
				&nbsp;<span class="label">[]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--consensus">--consensus </a>
			 / <small>-consensus</small>
		</h3>
		<p class="args">
			<b>1000G consensus mode</b><br />
			This argument is specifically intended for 1000G consensus analysis mode. Setting this flag will inject all
 provided alleles to the assembly graph but will not forcibly genotype all of them.
		</p>
		<p>
			<span class="label label-info ">boolean</span>
				&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--contamination_fraction_per_sample_file">--contamination_fraction_per_sample_file </a>
			 / <small>-contaminationFile</small>
		</h3>
		<p class="args">
			<b>Tab-separated File containing fraction of contamination in sequencing data (per sample) to aggressively remove. Format should be "<SampleID><TAB><Contamination>" (Contamination is double) per line; No header.</b><br />
			This argument specifies a file with two columns "sample" and "contamination" specifying the contamination level for those samples.
  Samples that do not appear in this file will be processed with CONTAMINATION_FRACTION.
		</p>
		<p>
			<span class="label label-info ">File</span>
				&nbsp;<span class="label">NA</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--contamination_fraction_to_filter">--contamination_fraction_to_filter </a>
			 / <small>-contamination</small>
		</h3>
		<p class="args">
			<b>Fraction of contamination in sequencing data (for all samples) to aggressively remove</b><br />
			If this fraction is greater is than zero, the caller will aggressively attempt to remove contamination through biased down-sampling of reads.
 Basically, it will ignore the contamination fraction of reads for each alternate allele.  So if the pileup contains N total bases, then we
 will try to remove (N * contamination fraction) bases for each alternate allele.
		</p>
		<p>
			<span class="label label-info ">double</span>
				&nbsp;<span class="label">0.0</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--cosmic">--cosmic </a>
			 / <small>-cosmic</small>
		</h3>
		<p class="args">
			<b>VCF file of COSMIC sites</b><br />
			MuTect2 has the ability to use COSMIC data in conjunction with dbSNP to adjust the threshold for evidence of a variant
 in the normal.  If a variant is present in dbSNP, but not in COSMIC, then more evidence is required from the normal
 sample to prove the variant is not present in germline.
		</p>
			<p>This argument supports reference-ordered data (ROD) files in the following formats: BCF2, VCF, VCF3</p>
		<p>
			<span class="label label-info ">List[RodBinding[VariantContext]]</span>
				&nbsp;<span class="label">[]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--dbsnp">--dbsnp </a>
			 / <small>-D</small>
		</h3>
		<p class="args">
			<b>dbSNP file</b><br />
			rsIDs from this file are used to populate the ID column of the output.  Also, the DB INFO flag will be set when appropriate.
 dbSNP overlap is only used to require more evidence of absence in the normal if the variant in question has been seen before in germline.
		</p>
			<p>This argument supports reference-ordered data (ROD) files in the following formats: BCF2, VCF, VCF3</p>
		<p>
			<span class="label label-info ">RodBinding[VariantContext]</span>
				&nbsp;<span class="label">none</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--dbsnp_normal_lod">--dbsnp_normal_lod </a>
			 / <small>NA</small>
		</h3>
		<p class="args">
			<b>LOD threshold for calling normal non-variant at dbsnp sites</b><br />
			The LOD threshold for the normal is typically made more strict if the variant has been seen in dbSNP (i.e. another
 normal sample). We thus require MORE evidence that a variant is NOT seen in this tumor's normal if it has been observed as a germline variant before.
		</p>
		<p>
			<span class="label label-info ">double</span>
				&nbsp;<span class="label">5.5</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--debug">--debug </a>
			 / <small>-debug</small>
		</h3>
		<p class="args">
			<b>Print out very verbose debug information about each triggering active region</b><br />
			
		</p>
		<p>
			<span class="label label-info ">boolean</span>
				&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--debug_read_name">--debug_read_name </a>
			 / <small>NA</small>
		</h3>
		<p class="args">
			<b>trace this read name through the calling process</b><br />
			
		</p>
		<p>
			<span class="label label-info ">String</span>
				&nbsp;<span class="label">NA</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--disableOptimizations">--disableOptimizations </a>
			 / <small>-disableOptimizations</small>
		</h3>
		<p class="args">
			<b>Don't skip calculations in ActiveRegions with no variants</b><br />
			If set, certain "early exit" optimizations in HaplotypeCaller, which aim to save compute and time by skipping
 calculations if an ActiveRegion is determined to contain no variants, will be disabled. This is most likely to be
 useful if you're using the <code>-bamout</code> argument to examine the placement of reads following reassembly
 and are interested in seeing the mapping of reads in regions with no variations. Setting the <code>-forceActive</code>
 and <code>-dontTrimActiveRegions</code> flags may also be helpful.
		</p>
		<p>
			<span class="label label-info ">boolean</span>
				&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--doNotRunPhysicalPhasing">--doNotRunPhysicalPhasing </a>
			 / <small>-doNotRunPhysicalPhasing</small>
		</h3>
		<p class="args">
			<b>Disable physical phasing</b><br />
			As of GATK 3.3, HaplotypeCaller outputs physical (read-based) information (see version 3.3 release notes and documentation for details). This argument disables that behavior.
		</p>
		<p>
			<span class="label label-info ">boolean</span>
				&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--dontIncreaseKmerSizesForCycles">--dontIncreaseKmerSizesForCycles </a>
			 / <small>-dontIncreaseKmerSizesForCycles</small>
		</h3>
		<p class="args">
			<b>Disable iterating over kmer sizes when graph cycles are detected</b><br />
			When graph cycles are detected, the normal behavior is to increase kmer sizes iteratively until the cycles are
 resolved. Disabling this behavior may cause the program to give up on assembling the ActiveRegion.
		</p>
		<p>
			<span class="label label-info ">boolean</span>
				&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--dontTrimActiveRegions">--dontTrimActiveRegions </a>
			 / <small>-dontTrimActiveRegions</small>
		</h3>
		<p class="args">
			<b>If specified, we will not trim down the active region from the full region (active + extension) to just the active interval for genotyping</b><br />
			
		</p>
		<p>
			<span class="label label-info ">boolean</span>
				&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--dontUseSoftClippedBases">--dontUseSoftClippedBases </a>
			 / <small>-dontUseSoftClippedBases</small>
		</h3>
		<p class="args">
			<b>If specified, we will not analyze soft clipped bases in the reads</b><br />
			
		</p>
		<p>
			<span class="label label-info ">boolean</span>
				&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--emitDroppedReads">--emitDroppedReads </a>
			 / <small>-edr</small>
		</h3>
		<p class="args">
			<b>Emit reads that are dropped for filtering, trimming, realignment failure</b><br />
			Determines whether dropped reads will be tracked and emitted when <code>-bamout</code> is specified. Use this in combination
 with a specific interval of interest to avoid accumulating a large number of reads in the <code>-bamout</code> file.
		</p>
		<p>
			<span class="label label-info ">boolean</span>
				&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--emitRefConfidence">--emitRefConfidence </a>
			 / <small>-ERC</small>
		</h3>
		<p class="args">
			<b>Mode for emitting reference confidence scores</b><br />
			The reference confidence mode makes it possible to emit variant calls in GVCF format, which includes either a per-base
 pair (BP_RESOLUTION) or a summarized (GVCF) confidence estimate for each position being strictly homozygous-reference.
 See http://www.broadinstitute.org/gatk/guide/article?id=2940 for more details of how this works.
 Note that if you use <code>-ERC</code> to emit a <code>GVCF</code> or <code>BP_RESOLUTION</code> output, you either
 need to give the output file the extension <code>.g.vcf</code> or set the parameters <code>-variant_index_type LINEAR</code>
 and <code>-variant_index_parameter 128000</code> (with those exact values!). This has to do with index compression.
		</p>
			<p>
			The --emitRefConfidence argument is an enumerated type (ReferenceConfidenceMode), which can have one of the following values:
			<dl class="enum">
					<dt class="enum">NONE</dt>
					<dd class="enum">Regular calling without emitting reference confidence calls.</dd>
					<dt class="enum">BP_RESOLUTION</dt>
					<dd class="enum">Reference model emitted site by site.</dd>
					<dt class="enum">GVCF</dt>
					<dd class="enum">Reference model emitted with condensed non-variant blocks, i.e. the GVCF format.</dd>
			</dl>
			</p>
		<p>
			<span class="label label-info ">ReferenceConfidenceMode</span>
				&nbsp;<span class="label">NONE</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--excludeAnnotation">--excludeAnnotation </a>
			 / <small>-XA</small>
		</h3>
		<p class="args">
			<b>One or more specific annotations to exclude</b><br />
			Which annotations to exclude from output in the VCF file.  Note that this argument has higher priority than the -A or -G arguments,
 so annotations will be excluded even if they are explicitly included with the other options.
		</p>
		<p>
			<span class="label label-info ">List[String]</span>
				&nbsp;<span class="label">[SpanningDeletions]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--forceActive">--forceActive </a>
			 / <small>-forceActive</small>
		</h3>
		<p class="args">
			<b>If provided, all bases will be tagged as active</b><br />
			For the active region walker to treat all bases as active.  Useful for debugging when you want to force something like
 the HaplotypeCaller to process a specific interval you provide the GATK
		</p>
		<p>
			<span class="label label-info ">boolean</span>
				&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--gcpHMM">--gcpHMM </a>
			 / <small>-gcpHMM</small>
		</h3>
		<p class="args">
			<b>Flat gap continuation penalty for use in the Pair HMM</b><br />
			
		</p>
		<p>
			<span class="label label-info ">int</span>
				&nbsp;<span class="label">10</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--genotyping_mode">--genotyping_mode </a>
			 / <small>-gt_mode</small>
		</h3>
		<p class="args">
			<b>Specifies how to determine the alternate alleles to use for genotyping</b><br />
			
		</p>
			<p>
			The --genotyping_mode argument is an enumerated type (GenotypingOutputMode), which can have one of the following values:
			<dl class="enum">
					<dt class="enum">DISCOVERY</dt>
					<dd class="enum">The genotyper will choose the most likely alternate allele</dd>
					<dt class="enum">GENOTYPE_GIVEN_ALLELES</dt>
					<dd class="enum">Only the alleles passed by the user should be considered.</dd>
			</dl>
			</p>
		<p>
			<span class="label label-info ">GenotypingOutputMode</span>
				&nbsp;<span class="label">DISCOVERY</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--graphOutput">--graphOutput </a>
			 / <small>-graph</small>
		</h3>
		<p class="args">
			<b>Write debug assembly graph information to this file</b><br />
			This argument is meant for debugging and is not immediately useful for normal analysis use.
		</p>
		<p>
			<span class="label label-info ">PrintStream</span>
				&nbsp;<span class="label">NA</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--group">--group </a>
			 / <small>-G</small>
		</h3>
		<p class="args">
			<b>One or more classes/groups of annotations to apply to variant calls</b><br />
			Which groups of annotations to add to the output VCF file. See the VariantAnnotator -list argument to view available groups.
		</p>
		<p>
			<span class="label label-info ">String[]</span>
				&nbsp;<span class="label">[]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--heterozygosity">--heterozygosity </a>
			 / <small>-hets</small>
		</h3>
		<p class="args">
			<b>Heterozygosity value used to compute prior likelihoods for any locus</b><br />
			The expected heterozygosity value used to compute prior probability that a locus is non-reference.

 From the heterozygosity we calculate the probability of N samples being hom-ref at a site as 1 - sum_i_2N (hets / i)
 where hets is this case is analogous to the parameter theta from population genetics. See https://en.wikipedia.org/wiki/Coalescent_theory for more details.

 Note that heterozygosity as used here is the population genetics concept. (See http://en.wikipedia.org/wiki/Zygosity#Heterozygosity_in_population_genetics. 
 We also suggest the book "Population Genetics: A Concise Guide" by John H. Gillespie for further details on the theory.) That is, a hets value of 0.001 
 implies that two randomly chosen chromosomes from the population of organisms would differ from each other at a rate of 1 in 1000 bp.

 The default priors provided for humans (hets = 1e-3)

 Also note that this quantity has nothing to do with the likelihood of any given sample having a heterozygous genotype,
 which in the GATK is purely determined by the probability of the observed data P(D | AB) under the model that there
 may be a AB het genotype.  The posterior probability of this AB genotype would use the het prior, but the GATK
 only uses this posterior probability in determining the prob. that a site is polymorphic.  So changing the
 het parameters only increases the chance that a site will be called non-reference across all samples, but
 doesn't actually change the output genotype likelihoods at all, as these aren't posterior probabilities at all.

 The quantity that changes whether the GATK considers the possibility of a het genotype at all is the ploidy,
 which determines how many chromosomes each individual in the species carries.
		</p>
		<p>
			<span class="label label-info ">Double</span>
				&nbsp;<span class="label">0.001</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--indel_heterozygosity">--indel_heterozygosity </a>
			 / <small>-indelHeterozygosity</small>
		</h3>
		<p class="args">
			<b>Heterozygosity for indel calling</b><br />
			This argument informs the prior probability of having an indel at a site.
		</p>
		<p>
			<span class="label label-info ">double</span>
				&nbsp;<span class="label">1.25E-4</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--initial_normal_lod">--initial_normal_lod </a>
			 / <small>NA</small>
		</h3>
		<p class="args">
			<b>Initial LOD threshold for calling normal variant</b><br />
			This is the LOD threshold corresponding to the minimum amount of reference evidence in the normal for a variant to be considered somatic and emitted in the VCF
		</p>
		<p>
			<span class="label label-info ">double</span>
				&nbsp;<span class="label">0.5</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--initial_tumor_lod">--initial_tumor_lod </a>
			 / <small>NA</small>
		</h3>
		<p class="args">
			<b>Initial LOD threshold for calling tumor variant</b><br />
			This is the LOD threshold that a variant must pass in the tumor to be emitted to the VCF. Note that the variant may pass this threshold yet still be annotated as FILTERed based on other criteria.
		</p>
		<p>
			<span class="label label-info ">double</span>
				&nbsp;<span class="label">4.0</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--input_prior">--input_prior </a>
			 / <small>-inputPrior</small>
		</h3>
		<p class="args">
			<b>Input prior for calls</b><br />
			By default, the prior specified with the argument --heterozygosity/-hets is used for variant discovery at a particular locus, using an infinite sites model,
 see e.g. Waterson (1975) or Tajima (1996).
 This model asserts that the probability of having a population of k variant sites in N chromosomes is proportional to theta/k, for 1=1:N

 There are instances where using this prior might not be desirable, e.g. for population studies where prior might not be appropriate,
 as for example when the ancestral status of the reference allele is not known.
 By using this argument, the user can manually specify a list of probabilities for each AC>1 to be used as priors for genotyping,
 with the following restrictions:
 a) User must specify 2N values, where N is the number of samples.
 b) Only diploid calls supported.
 c) Probability values are specified in Double format, in linear space (not log10 space or Phred-scale).
 d) No negative values allowed.
 e) Values will be added and Pr(AC=0) will be 1-sum, so that they sum up to one.
 f) If user-defined values add to more than one, an error will be produced.

 If user wants completely flat priors, then user should specify the same value (=1/(2*N+1)) 2*N times,e.g.
   -inputPrior 0.33 -inputPrior 0.33
 for the single-sample diploid case.
		</p>
		<p>
			<span class="label label-info ">List[Double]</span>
				&nbsp;<span class="label">[]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--kmerSize">--kmerSize </a>
			 / <small>-kmerSize</small>
		</h3>
		<p class="args">
			<b>Kmer size to use in the read threading assembler</b><br />
			Multiple kmer sizes can be specified, using e.g. `-kmerSize 10 -kmerSize 25`.
		</p>
		<p>
			<span class="label label-info ">List[Integer]</span>
				&nbsp;<span class="label">[10, 25]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--m2debug">--m2debug </a>
			 / <small>-m2debug</small>
		</h3>
		<p class="args">
			<b>Print out very verbose M2 debug information</b><br />
			
		</p>
		<p>
			<span class="label label-info ">boolean</span>
				&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--max_alt_allele_in_normal_fraction">--max_alt_allele_in_normal_fraction </a>
			 / <small>NA</small>
		</h3>
		<p class="args">
			<b>Threshold for maximum alternate allele fraction in normal</b><br />
			This argument is used for the internal "alt_allele_in_normal" filter.
 A variant will PASS the filter if the value tested is lower or equal to the threshold value. It will FAIL the filter if the value tested is greater than the max threshold value.
		</p>
		<p>
			<span class="label label-info ">double</span>
				&nbsp;<span class="label">0.03</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--max_alt_alleles_in_normal_count">--max_alt_alleles_in_normal_count </a>
			 / <small>NA</small>
		</h3>
		<p class="args">
			<b>Threshold for maximum alternate allele counts in normal</b><br />
			This argument is used for the internal "alt_allele_in_normal" filter.
 A variant will PASS the filter if the value tested is lower or equal to the threshold value. It will FAIL the filter if the value tested is greater than the max threshold value.
		</p>
		<p>
			<span class="label label-info ">int</span>
				&nbsp;<span class="label">1</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--max_alt_alleles_in_normal_qscore_sum">--max_alt_alleles_in_normal_qscore_sum </a>
			 / <small>NA</small>
		</h3>
		<p class="args">
			<b>Threshold for maximum alternate allele quality score sum in normal</b><br />
			This argument is used for the internal "alt_allele_in_normal" filter.
 A variant will PASS the filter if the value tested is lower or equal to the threshold value. It will FAIL the filter if the value tested is greater than the max threshold value.
		</p>
		<p>
			<span class="label label-info ">int</span>
				&nbsp;<span class="label">20</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--max_alternate_alleles">--max_alternate_alleles </a>
			 / <small>-maxAltAlleles</small>
		</h3>
		<p class="args">
			<b>Maximum number of alternate alleles to genotype</b><br />
			If there are more than this number of alternate alleles presented to the genotyper (either through discovery or GENOTYPE_GIVEN_ALLELES),
 then only this many alleles will be used.  Note that genotyping sites with many alternate alleles is both CPU and memory intensive and it
 scales exponentially based on the number of alternate alleles.  Unless there is a good reason to change the default value, we highly recommend
 that you not play around with this parameter.

 As of GATK 2.2 the genotyper can handle a very large number of events, so the default maximum has been increased to 6.
		</p>
		<p>
			<span class="label label-info ">int</span>
				&nbsp;<span class="label">6</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--max_num_PL_values">--max_num_PL_values </a>
			 / <small>-maxNumPLValues</small>
		</h3>
		<p class="args">
			<b>Maximum number of PL values to output</b><br />
			Determines the maximum number of PL values that will be logged in the output.  If the number of genotypes
 (which is determined by the ploidy and the number of alleles) exceeds the value provided by this argument,
 then output of all of the PL values will be suppressed.
		</p>
		<p>
			<span class="label label-info ">int</span>
				&nbsp;<span class="label">100</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--maxNumHaplotypesInPopulation">--maxNumHaplotypesInPopulation </a>
			 / <small>-maxNumHaplotypesInPopulation</small>
		</h3>
		<p class="args">
			<b>Maximum number of haplotypes to consider for your population</b><br />
			The assembly graph can be quite complex, and could imply a very large number of possible haplotypes.  Each haplotype
 considered requires N PairHMM evaluations if there are N reads across all samples.  In order to control the
 run of the haplotype caller we only take maxNumHaplotypesInPopulation paths from the graph, in order of their
 weights, no matter how many paths are possible to generate from the graph.  Putting this number too low
 will result in dropping true variation because paths that include the real variant are not even considered.
 You can consider increasing this number when calling organisms with high heterozygosity.
		</p>
		<p>
			<span class="label label-info ">int</span>
				&nbsp;<span class="label">128</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--min_base_quality_score">--min_base_quality_score </a>
			 / <small>-mbq</small>
		</h3>
		<p class="args">
			<b>Minimum base quality required to consider a base for calling</b><br />
			The minimum confidence needed for a given base for it to be used in variant calling.
		</p>
		<p>
			<span class="label label-info ">byte</span>
				&nbsp;<span class="label">10</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--minDanglingBranchLength">--minDanglingBranchLength </a>
			 / <small>-minDanglingBranchLength</small>
		</h3>
		<p class="args">
			<b>Minimum length of a dangling branch to attempt recovery</b><br />
			When constructing the assembly graph we are often left with "dangling" branches.  The assembly engine attempts to rescue these branches
 by merging them back into the main graph.  This argument describes the minimum length of a dangling branch needed for the engine to
 try to rescue it.  A smaller number here will lead to higher sensitivity to real variation but also to a higher number of false positives.
		</p>
		<p>
			<span class="label label-info ">int</span>
				&nbsp;<span class="label">4</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--minPruning">--minPruning </a>
			 / <small>-minPruning</small>
		</h3>
		<p class="args">
			<b>Minimum support to not prune paths in the graph</b><br />
			Paths with fewer supporting kmers than the specified threshold will be pruned from the graph.

 Be aware that this argument can dramatically affect the results of variant calling and should only be used with great caution.
 Using a prune factor of 1 (or below) will prevent any pruning from the graph, which is generally not ideal; it can make the
 calling much slower and even less accurate (because it can prevent effective merging of "tails" in the graph).  Higher values
 tend to make the calling much faster, but also lowers the sensitivity of the results (because it ultimately requires higher
 depth to produce calls).
		</p>
		<p>
			<span class="label label-info ">int</span>
				&nbsp;<span class="label">2</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--normal_lod">--normal_lod </a>
			 / <small>NA</small>
		</h3>
		<p class="args">
			<b>LOD threshold for calling normal non-germline</b><br />
			This is a measure of the minimum evidence to support that a variant observed in the tumor is not also present in the normal.
		</p>
		<p>
			<span class="label label-info ">double</span>
				&nbsp;<span class="label">2.2</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--normal_panel">--normal_panel </a>
			 / <small>-PON</small>
		</h3>
		<p class="args">
			<b>VCF file of sites observed in normal</b><br />
			A panel of normals can be a useful (optional) input to help filter out commonly seen sequencing noise that may appear as low allele-fraction somatic variants.
		</p>
			<p>This argument supports reference-ordered data (ROD) files in the following formats: BCF2, VCF, VCF3</p>
		<p>
			<span class="label label-info ">List[RodBinding[VariantContext]]</span>
				&nbsp;<span class="label">[]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--numPruningSamples">--numPruningSamples </a>
			 / <small>-numPruningSamples</small>
		</h3>
		<p class="args">
			<b>Number of samples that must pass the minPruning threshold</b><br />
			If fewer samples than the specified number pass the minPruning threshold for a given path, that path will be eliminated from the graph.
		</p>
		<p>
			<span class="label label-info ">int</span>
				&nbsp;<span class="label">1</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--out">--out </a>
			 / <small>-o</small>
		</h3>
		<p class="args">
			<b>File to which variants should be written</b><br />
			A raw, unfiltered, highly sensitive callset in VCF format.
		</p>
		<p>
			<span class="label label-info ">VariantContextWriter</span>
				&nbsp;<span class="label">stdout</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--output_mode">--output_mode </a>
			 / <small>-out_mode</small>
		</h3>
		<p class="args">
			<b>Specifies which type of calls we should output</b><br />
			
		</p>
			<p>
			The --output_mode argument is an enumerated type (OutputMode), which can have one of the following values:
			<dl class="enum">
					<dt class="enum">EMIT_VARIANTS_ONLY</dt>
					<dd class="enum">produces calls only at variant sites</dd>
					<dt class="enum">EMIT_ALL_CONFIDENT_SITES</dt>
					<dd class="enum">produces calls at variant sites and confident reference sites</dd>
					<dt class="enum">EMIT_ALL_SITES</dt>
					<dd class="enum">produces calls at any callable site regardless of confidence; this argument is intended only for point
 mutations (SNPs) in DISCOVERY mode or generally when running in GENOTYPE_GIVEN_ALLELES mode; it will by
 no means produce a comprehensive set of indels in DISCOVERY mode</dd>
			</dl>
			</p>
		<p>
			<span class="label label-info ">OutputMode</span>
				&nbsp;<span class="label">EMIT_VARIANTS_ONLY</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--phredScaledGlobalReadMismappingRate">--phredScaledGlobalReadMismappingRate </a>
			 / <small>-globalMAPQ</small>
		</h3>
		<p class="args">
			<b>The global assumed mismapping rate for reads</b><br />
			The phredScaledGlobalReadMismappingRate reflects the average global mismapping rate of all reads, regardless of their
 mapping quality.  This term effects the probability that a read originated from the reference haplotype, regardless of
 its edit distance from the reference, in that the read could have originated from the reference haplotype but
 from another location in the genome.  Suppose a read has many mismatches from the reference, say like 5, but
 has a very high mapping quality of 60.  Without this parameter, the read would contribute 5 * Q30 evidence
 in favor of its 5 mismatch haplotype compared to reference, potentially enough to make a call off that single
 read for all of these events.  With this parameter set to Q30, though, the maximum evidence against any haplotype
 that this (and any) read could contribute is Q30.

 Set this term to any negative number to turn off the global mapping rate.
		</p>
		<p>
			<span class="label label-info ">int</span>
				&nbsp;<span class="label">45</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--sample_ploidy">--sample_ploidy </a>
			 / <small>-ploidy</small>
		</h3>
		<p class="args">
			<b>Ploidy (number of chromosomes) per sample. For pooled data, set to (Number of samples in each pool * Sample Ploidy).</b><br />
			Sample ploidy - equivalent to number of chromosomes per pool. In pooled experiments this should be = # of samples in pool * individual sample ploidy
		</p>
		<p>
			<span class="label label-info ">int</span>
				&nbsp;<span class="label">2</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--standard_min_confidence_threshold_for_calling">--standard_min_confidence_threshold_for_calling </a>
			 / <small>-stand_call_conf</small>
		</h3>
		<p class="args">
			<b>The minimum phred-scaled confidence threshold at which variants should be called</b><br />
			The minimum phred-scaled Qscore threshold to separate high confidence from low confidence calls. Only genotypes with
 confidence >= this threshold are emitted as called sites. A reasonable threshold is 30 for high-pass calling (this
 is the default).
		</p>
		<p>
			<span class="label label-info ">double</span>
				&nbsp;<span class="label">30.0</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--standard_min_confidence_threshold_for_emitting">--standard_min_confidence_threshold_for_emitting </a>
			 / <small>-stand_emit_conf</small>
		</h3>
		<p class="args">
			<b>The minimum phred-scaled confidence threshold at which variants should be emitted (and filtered with LowQual if less than the calling threshold)</b><br />
			This argument allows you to emit low quality calls as filtered records.
		</p>
		<p>
			<span class="label label-info ">double</span>
				&nbsp;<span class="label">30.0</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--tumor_lod">--tumor_lod </a>
			 / <small>NA</small>
		</h3>
		<p class="args">
			<b>LOD threshold for calling tumor variant</b><br />
			Only variants with tumor LODs exceeding this threshold can pass filtering.
		</p>
		<p>
			<span class="label label-info ">double</span>
				&nbsp;<span class="label">6.3</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--useFilteredReadsForAnnotations">--useFilteredReadsForAnnotations </a>
			 / <small>-useFilteredReadsForAnnotations</small>
		</h3>
		<p class="args">
			<b>Use the contamination-filtered read maps for the purposes of annotating variants</b><br />
			
		</p>
		<p>
			<span class="label label-info ">boolean</span>
				&nbsp;<span class="label">false</span>
		</p>

        <hr>
        <p><a href='#top'><i class='fa fa-chevron-up'></i> Return to top</a></p>
        <hr>
        <p class="see-also">See also 
        	<a href="http://software.broadinstitute.org/gatk/documentation/">GATK Documentation Index</a> |
        	<a href="index.html">Tool Docs Index</a> |
        	<a href="http://gatkforums.broadinstitute.org/gatk">Support Forum</a>
        </p>

        <p class="version">GATK version 3.6-0-g89b7209 built at 2017/02/09 12:52:48.
        </p>

	</div>

</div>

