

<div class='row' id="top">

	

	

	<section class="col-md-4">
		<aside class="well">
			<a href="index.html"><h4><i class='fa fa-chevron-left'></i> Back to Tool Docs Index</h4></a>
		</aside>
		<aside class="well">
			<h2>Categories</h2>

        <ul class="nav nav-pills nav-stacked" id="sidenav">
        		<hr>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#EngineParametersavailabletoalltools">Engine Parameters (available to all tools)</a>
							<div id="EngineParametersavailabletoalltools"
								
								<ul>
											<li>
												<a href="org_broadinstitute_gatk_engine_CommandLineGATK.html">CommandLineGATK</a>
											</li>
								</ul>
							</div>
						</li>
        		<hr>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#DiagnosticsandQualityControlTools">Diagnostics and Quality Control Tools</a>
							<div id="DiagnosticsandQualityControlTools"
								
								<ul>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_rnaseq_ASEReadCounter.html">ASEReadCounter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_bqsr_AnalyzeCovariates.html">AnalyzeCovariates</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_coverage_CallableLoci.html">CallableLoci</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_CheckPileup.html">CheckPileup</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_coverage_CompareCallableLoci.html">CompareCallableLoci</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_cancer_contamination_ContEst.html">ContEst</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_CountBases.html">CountBases</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_CountIntervals.html">CountIntervals</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_CountLoci.html">CountLoci</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_CountMales.html">CountMales</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_CountRODs.html">CountRODs</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_CountRODsByRef.html">CountRODsByRef</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_CountReadEvents.html">CountReadEvents</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_CountReads.html">CountReads</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_CountTerminusEvent.html">CountTerminusEvent</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_coverage_DepthOfCoverage.html">DepthOfCoverage</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_diagnostics_diagnosetargets_DiagnoseTargets.html">DiagnoseTargets</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_diffengine_DiffObjects.html">DiffObjects</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_diagnostics_ErrorRatePerCycle.html">ErrorRatePerCycle</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_fasta_FastaStats.html">FastaStats</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_diagnostics_FindCoveredIntervals.html">FindCoveredIntervals</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_FlagStat.html">FlagStat</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_coverage_GCContentByInterval.html">GCContentByInterval</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_GatherBqsrReports.html">GatherBqsrReports</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_Pileup.html">Pileup</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_PrintRODs.html">PrintRODs</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_diagnostics_missing_QualifyMissingIntervals.html">QualifyMissingIntervals</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_ReadClippingStats.html">ReadClippingStats</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_diagnostics_ReadGroupProperties.html">ReadGroupProperties</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_diagnostics_ReadLengthDistribution.html">ReadLengthDistribution</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_simulatereads_SimulateReadsForVariants.html">SimulateReadsForVariants</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#SequenceDataProcessingTools">Sequence Data Processing Tools</a>
							<div id="SequenceDataProcessingTools"
								
								<ul>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_bqsr_BaseRecalibrator.html">BaseRecalibrator</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_readutils_ClipReads.html">ClipReads</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_indels_IndelRealigner.html">IndelRealigner</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_indels_LeftAlignIndels.html">LeftAlignIndels</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_readutils_PrintReads.html">PrintReads</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_indels_RealignerTargetCreator.html">RealignerTargetCreator</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_rnaseq_SplitNCigarReads.html">SplitNCigarReads</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_readutils_SplitSamFile.html">SplitSamFile</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#VariantDiscoveryTools">Variant Discovery Tools</a>
							<div id="VariantDiscoveryTools"
								
								<ul>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantrecalibration_ApplyRecalibration.html">ApplyRecalibration</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_CalculateGenotypePosteriors.html">CalculateGenotypePosteriors</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_examples_GATKPaperGenotyper.html">GATKPaperGenotyper</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_GenotypeGVCFs.html">GenotypeGVCFs</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_haplotypecaller_HaplotypeCaller.html">HaplotypeCaller</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_cancer_m2_MuTect2.html">MuTect2</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_RegenotypeVariants.html">RegenotypeVariants</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_genotyper_UnifiedGenotyper.html">UnifiedGenotyper</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantrecalibration_VariantRecalibrator.html">VariantRecalibrator</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#VariantEvaluationTools">Variant Evaluation Tools</a>
							<div id="VariantEvaluationTools"
								
								<ul>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_GenotypeConcordance.html">GenotypeConcordance</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_ValidateVariants.html">ValidateVariants</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_varianteval_VariantEval.html">VariantEval</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_filters_VariantFiltration.html">VariantFiltration</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#VariantManipulationTools">Variant Manipulation Tools</a>
							<div id="VariantManipulationTools"
								
								<ul>
											<li>
												<a href="org_broadinstitute_gatk_tools_CatVariants.html">CatVariants</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_CombineGVCFs.html">CombineGVCFs</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_CombineVariants.html">CombineVariants</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_haplotypecaller_HaplotypeResolver.html">HaplotypeResolver</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_LeftAlignAndTrimVariants.html">LeftAlignAndTrimVariants</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_phasing_PhaseByTransmission.html">PhaseByTransmission</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_RandomlySplitVariants.html">RandomlySplitVariants</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_phasing_ReadBackedPhasing.html">ReadBackedPhasing</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_SelectHeaders.html">SelectHeaders</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_SelectVariants.html">SelectVariants</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_validation_validationsiteselector_ValidationSiteSelector.html">ValidationSiteSelector</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_VariantAnnotator.html">VariantAnnotator</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_VariantsToAllelicPrimitives.html">VariantsToAllelicPrimitives</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_VariantsToBinaryPed.html">VariantsToBinaryPed</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_VariantsToTable.html">VariantsToTable</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_variantutils_VariantsToVCF.html">VariantsToVCF</a>
											</li>
								</ul>
							</div>
						</li>
        		<hr>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#AnnotationModules">Annotation Modules</a>
							<div id="AnnotationModules"
								
								<ul>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AS_BaseQualityRankSumTest.html">AS_BaseQualityRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AS_FisherStrand.html">AS_FisherStrand</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AS_InbreedingCoeff.html">AS_InbreedingCoeff</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AS_InsertSizeRankSum.html">AS_InsertSizeRankSum</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AS_MQMateRankSumTest.html">AS_MQMateRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AS_MappingQualityRankSumTest.html">AS_MappingQualityRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AS_QualByDepth.html">AS_QualByDepth</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AS_RMSMappingQuality.html">AS_RMSMappingQuality</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AS_ReadPosRankSumTest.html">AS_ReadPosRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AS_StrandOddsRatio.html">AS_StrandOddsRatio</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AlleleBalance.html">AlleleBalance</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AlleleBalanceBySample.html">AlleleBalanceBySample</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_AlleleCountBySample.html">AlleleCountBySample</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_BaseCounts.html">BaseCounts</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_BaseCountsBySample.html">BaseCountsBySample</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_BaseQualityRankSumTest.html">BaseQualityRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_cancer_BaseQualitySumPerAlleleBySample.html">BaseQualitySumPerAlleleBySample</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_ChromosomeCounts.html">ChromosomeCounts</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_ClippingRankSumTest.html">ClippingRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_cancer_ClusteredReadPosition.html">ClusteredReadPosition</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_Coverage.html">Coverage</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_DepthPerAlleleBySample.html">DepthPerAlleleBySample</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_DepthPerSampleHC.html">DepthPerSampleHC</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_ExcessHet.html">ExcessHet</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_FisherStrand.html">FisherStrand</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_FractionInformativeReads.html">FractionInformativeReads</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_GCContent.html">GCContent</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_GenotypeSummaries.html">GenotypeSummaries</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_HaplotypeScore.html">HaplotypeScore</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_HardyWeinberg.html">HardyWeinberg</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_HomopolymerRun.html">HomopolymerRun</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_InbreedingCoeff.html">InbreedingCoeff</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_LikelihoodRankSumTest.html">LikelihoodRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_LowMQ.html">LowMQ</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_MVLikelihoodRatio.html">MVLikelihoodRatio</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_MappingQualityRankSumTest.html">MappingQualityRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_MappingQualityZero.html">MappingQualityZero</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_MappingQualityZeroBySample.html">MappingQualityZeroBySample</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_NBaseCount.html">NBaseCount</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_cancer_OxoGReadCounts.html">OxoGReadCounts</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_PossibleDeNovo.html">PossibleDeNovo</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_QualByDepth.html">QualByDepth</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_RMSMappingQuality.html">RMSMappingQuality</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_ReadPosRankSumTest.html">ReadPosRankSumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_SampleList.html">SampleList</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_SnpEff.html">SnpEff</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_SpanningDeletions.html">SpanningDeletions</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_StrandAlleleCountsBySample.html">StrandAlleleCountsBySample</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_StrandBiasBySample.html">StrandBiasBySample</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_StrandOddsRatio.html">StrandOddsRatio</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_TandemRepeatAnnotator.html">TandemRepeatAnnotator</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_TransmissionDisequilibriumTest.html">TransmissionDisequilibriumTest</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_annotator_VariantType.html">VariantType</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#ReadFilters">Read Filters</a>
							<div id="ReadFilters"
								
								<ul>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_BadCigarFilter.html">BadCigarFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_BadMateFilter.html">BadMateFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_CountingFilteringIterator$CountingReadFilter.html">CountingFilteringIterator.CountingReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_DuplicateReadFilter.html">DuplicateReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_FailsVendorQualityCheckFilter.html">FailsVendorQualityCheckFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_haplotypecaller_HCMappingQualityFilter.html">HCMappingQualityFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_LibraryReadFilter.html">LibraryReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_MalformedReadFilter.html">MalformedReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_MappingQualityFilter.html">MappingQualityFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_MappingQualityUnavailableFilter.html">MappingQualityUnavailableFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_MappingQualityZeroFilter.html">MappingQualityZeroFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_MateSameStrandFilter.html">MateSameStrandFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_MaxInsertSizeFilter.html">MaxInsertSizeFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_MissingReadGroupFilter.html">MissingReadGroupFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_NoOriginalQualityScoresFilter.html">NoOriginalQualityScoresFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_NotPrimaryAlignmentFilter.html">NotPrimaryAlignmentFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_OverclippedReadFilter.html">OverclippedReadFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_Platform454Filter.html">Platform454Filter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_PlatformFilter.html">PlatformFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_PlatformUnitFilter.html">PlatformUnitFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_ReadGroupBlackListFilter.html">ReadGroupBlackListFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_ReadLengthFilter.html">ReadLengthFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_ReadNameFilter.html">ReadNameFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_ReadStrandFilter.html">ReadStrandFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_ReassignMappingQualityFilter.html">ReassignMappingQualityFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_ReassignOneMappingQualityFilter.html">ReassignOneMappingQualityFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_ReassignOriginalMQAfterIndelRealignmentFilter.html">ReassignOriginalMQAfterIndelRealignmentFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_SampleFilter.html">SampleFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_SingleReadGroupFilter.html">SingleReadGroupFilter</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_engine_filters_UnmappedReadFilter.html">UnmappedReadFilter</a>
											</li>
								</ul>
							</div>
						</li>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#ResourceFileCodecs">Resource File Codecs</a>
							<div id="ResourceFileCodecs"
								
								<ul>
											<li>
												<a href="org_broadinstitute_gatk_utils_codecs_beagle_BeagleCodec.html">BeagleCodec</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_utils_codecs_table_BedTableCodec.html">BedTableCodec</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_utils_codecs_hapmap_RawHapMapCodec.html">RawHapMapCodec</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_utils_codecs_refseq_RefSeqCodec.html">RefSeqCodec</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_utils_codecs_sampileup_SAMPileupCodec.html">SAMPileupCodec</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_utils_codecs_samread_SAMReadCodec.html">SAMReadCodec</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_utils_codecs_table_TableCodec.html">TableCodec</a>
											</li>
								</ul>
							</div>
						</li>
        		<hr>
						<li><a data-toggle="collapse" data-parent="#sidenav" href="#ReferenceUtilities">Reference Utilities</a>
							<div id="ReferenceUtilities"
								
								<ul>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_fasta_FastaAlternateReferenceMaker.html">FastaAlternateReferenceMaker</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_fasta_FastaReferenceMaker.html">FastaReferenceMaker</a>
											</li>
											<li>
												<a href="org_broadinstitute_gatk_tools_walkers_qc_QCRef.html">QCRef</a>
											</li>
								</ul>
							</div>
						</li>
        </ul>
		</aside>
		

	</section>

	<div class="col-md-8">           <h1><small>Showing docs for version 3.7-0 </small></h1>     <hr />

		<h1>HaplotypeCaller</h1>

		<p class="lead">Call germline SNPs and indels via local re-assembly of haplotypes</p>

			<h3>Category
				<small> Variant Discovery Tools</small>
			</h3>
			<h3>Traversal
				<small>ActiveRegionWalker</small>
			</h3>
			<h3>PartitionBy
				<small>LOCUS</small>
			</h3>

		<hr>
		<h2>Overview</h2>
		

 <p>The HaplotypeCaller is capable of calling SNPs and indels simultaneously via local de-novo assembly of haplotypes in an active region. In other words, whenever the program encounters a region showing signs of variation, it discards the existing mapping information and completely reassembles the reads in that region. This allows the HaplotypeCaller to be more accurate when calling regions that are traditionally difficult to call, for example when they contain different types of variants close to each other. It also makes the HaplotypeCaller much better at calling indels than position-based callers like UnifiedGenotyper.</p>

<p>In the so-called GVCF mode used for scalable variant calling in DNA sequence data, HaplotypeCaller runs per-sample to generate an intermediate genomic gVCF (gVCF), which can then be used for joint genotyping of multiple samples in a very efficient way, which enables rapid incremental processing of samples as they roll off the sequencer, as well as scaling to very large cohort sizes (e.g. the 92K exomes of ExAC).</p>

 <p>In addition, HaplotypeCaller is able to handle non-diploid organisms as well as pooled experiment data. Note however that the algorithms used to calculate variant likelihoods is not well suited to extreme allele frequencies (relative to ploidy) so its use is not recommended for somatic (cancer) variant discovery. For that purpose, use MuTect2 instead.</p>

 <p>Finally, HaplotypeCaller is also able to correctly handle the splice junctions that make RNAseq a challenge for most variant callers.</p>

 <h3>How HaplotypeCaller works</h3>

 <br />
 <h4>1. Define active regions </h4>

 <p>The program determines which regions of the genome it needs to operate on, based on the presence of significant
 evidence for variation.</p>

 <br />
 <h4>2. Determine haplotypes by assembly of the active region </h4>

 <p>For each ActiveRegion, the program builds a De Bruijn-like graph to reassemble the ActiveRegion, and identifies
 what are the possible haplotypes present in the data. The program then realigns each haplotype against the reference
 haplotype using the Smith-Waterman algorithm in order to identify potentially variant sites. </p>

 <br />
 <h4>3. Determine likelihoods of the haplotypes given the read data </h4>

 <p>For each ActiveRegion, the program performs a pairwise alignment of each read against each haplotype using the
 PairHMM algorithm. This produces a matrix of likelihoods of haplotypes given the read data. These likelihoods are
 then marginalized to obtain the likelihoods of alleles for each potentially variant site given the read data.   </p>

 <br />
 <h4>4. Assign sample genotypes </h4>

 <p>For each potentially variant site, the program applies Bayes' rule, using the likelihoods of alleles given the
 read data to calculate the likelihoods of each genotype per sample given the read data observed for that
 sample. The most likely genotype is then assigned to the sample.    </p>

 <h3>Input</h3>
 <p>
 Input bam file(s) from which to make calls
 </p>

 <h3>Output</h3>
 <p>
 Either a VCF or gVCF file with raw, unfiltered SNP and indel calls. Regular VCFs must be filtered either by variant
 recalibration (best) or hard-filtering before use in downstream analyses. If using the reference-confidence model
 workflow for cohort analysis, the output is a GVCF file that must first be run through GenotypeGVCFs and then
 filtering before further analysis.
 </p>

 <h3>Usage examples</h3>

 <p>These are example commands that show how to run HaplotypeCaller for typical use cases. Square brackets ("[ ]")
 indicate optional arguments. Note that parameter values shown here may not be the latest recommended; see the
 Best Practices documentation for detailed recommendations. </p>

 <br />
 <h4>Single-sample GVCF calling on DNAseq (for `-ERC GVCF` cohort analysis workflow)</h4>
 <pre>
   java -jar GenomeAnalysisTK.jar \
     -R reference.fasta \
     -T HaplotypeCaller \
     -I sample1.bam \
     --emitRefConfidence GVCF \
     [--dbsnp dbSNP.vcf] \
     [-L targets.interval_list] \
     -o output.raw.snps.indels.g.vcf
 </pre>

 <h4>Single-sample GVCF calling on DNAseq with allele-specific annotations (for allele-specific cohort analysis workflow)</h4>
 <pre>
   java -jar GenomeAnalysisTK.jar \
     -R reference.fasta \
     -T HaplotypeCaller \
     -I sample1.bam \
     --emitRefConfidence GVCF \
     [--dbsnp dbSNP.vcf] \
     [-L targets.interval_list] \
     -G Standard -G AS_Standard \
     -o output.raw.snps.indels.AS.g.vcf
 </pre>

 <h4>Variant-only calling on DNAseq</h4>
 <pre>
   java -jar GenomeAnalysisTK.jar \
     -R reference.fasta \
     -T HaplotypeCaller \
     -I sample1.bam [-I sample2.bam ...] \
     [--dbsnp dbSNP.vcf] \
     [-stand_call_conf 30] \
     [-L targets.interval_list] \
     -o output.raw.snps.indels.vcf
 </pre>

 <h4>Variant-only calling on RNAseq</h4>
 <pre>
   java -jar GenomeAnalysisTK.jar \
     -R reference.fasta \
     -T HaplotypeCaller \
     -I sample1.bam \
     [--dbsnp dbSNP.vcf] \
     -stand_call_conf 20 \
     -o output.raw.snps.indels.vcf
 </pre>

 <h3>Caveats</h3>
 <ul>
 <li>We have not yet fully tested the interaction between the GVCF-based calling or the multisample calling and the
 RNAseq-specific functionalities. Use those in combination at your own risk.</li>
 <li>Many users have reported issues running HaplotypeCaller with the -nct argument, so we recommend using Queue to
 parallelize HaplotypeCaller instead of multithreading.</li>
 </ul>

 <h3>Special note on ploidy</h3>
 <p>This tool is able to handle almost any ploidy (except very high ploidies in large pooled experiments); the ploidy can be specified using the -ploidy argument for non-diploid organisms.</p>

 <h3>Additional Notes</h3>
 <ul>
     <li>When working with PCR-free data, be sure to set `-pcr_indel_model NONE` (see argument below).</li>
     <li>When running in `-ERC GVCF` or `-ERC BP_RESOLUTION` modes, the emitting and calling confidence thresholds
     are automatically set to 0. This cannot be overridden by the command line. The thresholds can be set manually
     to the desired levels in the next step of the workflow (GenotypeGVCFs)</li>
 </ul>

				<hr>
				<h2>Additional Information</h2>
				<p></p>
				<h3>Read filters</h3>
					<p>These Read Filters are automatically applied to the data by the Engine before processing by HaplotypeCaller.</p>
				<ul>
						<li><a href="org_broadinstitute_gatk_tools_walkers_haplotypecaller_HCMappingQualityFilter.html">HCMappingQualityFilter</a></li>
						<li><a href="org_broadinstitute_gatk_engine_filters_MalformedReadFilter.html">MalformedReadFilter</a></li>
						<li><a href="org_broadinstitute_gatk_engine_filters_BadCigarFilter.html">BadCigarFilter</a></li>
						<li><a href="org_broadinstitute_gatk_engine_filters_UnmappedReadFilter.html">UnmappedReadFilter</a></li>
						<li><a href="org_broadinstitute_gatk_engine_filters_NotPrimaryAlignmentFilter.html">NotPrimaryAlignmentFilter</a></li>
						<li><a href="org_broadinstitute_gatk_engine_filters_FailsVendorQualityCheckFilter.html">FailsVendorQualityCheckFilter</a></li>
						<li><a href="org_broadinstitute_gatk_engine_filters_DuplicateReadFilter.html">DuplicateReadFilter</a></li>
						<li><a href="org_broadinstitute_gatk_engine_filters_MappingQualityUnavailableFilter.html">MappingQualityUnavailableFilter</a></li>
				</ul>
				<h3>Parallelism options</h3>
					<p>This tool can be run in multi-threaded mode using this option.</p>
				<ul>
						<li><a href="https://software.broadinstitute.org/gatk/documentation/tooldocs/org_broadinstitute_gatk_engine_CommandLineGATK.html#-nct">NanoSchedulable (-nct)</a></li>
				</ul>
				<h3>Downsampling settings</h3>
					<p>This tool applies the following downsampling settings by default.</p>
					<ul>
						<li>Mode: <b>BY_SAMPLE</b></li>
						<li>To coverage: <b>500</b></li>
					</ul>
				<h3>ActiveRegion settings</h3>
				<p>This tool uses ActiveRegions on the reference.</p>
				<ul>
					<li>Minimum region size: <b>50</b> bp</li>
					<li>Maximum region size: <b>300</b> bp</li>
					<li>Extension increments: <b>100</b> bp</li>
				</ul>
				<hr>
				<h2>Command-line Arguments</h2>
				<p></p>
				<h3>Engine arguments</h3>
				<p>All tools inherit arguments from the GATK Engine' "CommandLineGATK" argument collection, which can be
					used to modify various aspects of the tool's function. For example, the -L argument directs the GATK
					engine to restrict processing to specific genomic intervals; or the -rf argument allows you to apply
					certain read filters to exclude some of the data from the analysis.</p>
				<ul>
						<li><a href="org_broadinstitute_gatk_engine_CommandLineGATK.html">CommandLineGATK</a></li>
				</ul>


					<h3>HaplotypeCaller specific arguments</h3>
					<p>This table summarizes the command-line arguments that are specific to this tool. For more details on each argument, see the list further down below the table or click on an argument name to jump directly to that entry in the list.</p>
					<table class="table table-striped table-bordered table-condensed">
						<thead>
						<tr>
							<th>Argument name(s)</th>
							<th>Default value</th>
							<th>Summary</th>
						</tr>
						</thead>
						<tbody>
			<tr>
				<th colspan="4" id="row-divider">Optional Inputs</th>
			</tr>
				<tr>
					<td><a href="#--alleles">--alleles</a><br />
					</td>
					<!--<td>RodBinding[VariantContext]</td> -->
					<td>none</td>
					<td>Set of alleles to use in genotyping</td>
				</tr>
				<tr>
					<td><a href="#--dbsnp">--dbsnp</a><br />
								&nbsp;<em>-D</em>
					</td>
					<!--<td>RodBinding[VariantContext]</td> -->
					<td>none</td>
					<td>dbSNP file</td>
				</tr>
			<tr>
				<th colspan="4" id="row-divider">Optional Outputs</th>
			</tr>
				<tr>
					<td><a href="#--activeRegionOut">--activeRegionOut</a><br />
								&nbsp;<em>-ARO</em>
					</td>
					<!--<td>PrintStream</td> -->
					<td>NA</td>
					<td>Output the active region to this IGV formatted file</td>
				</tr>
				<tr>
					<td><a href="#--activityProfileOut">--activityProfileOut</a><br />
								&nbsp;<em>-APO</em>
					</td>
					<!--<td>PrintStream</td> -->
					<td>NA</td>
					<td>Output the raw activity profile results in IGV format</td>
				</tr>
				<tr>
					<td><a href="#--graphOutput">--graphOutput</a><br />
								&nbsp;<em>-graph</em>
					</td>
					<!--<td>PrintStream</td> -->
					<td>NA</td>
					<td>Write debug assembly graph information to this file</td>
				</tr>
				<tr>
					<td><a href="#--out">--out</a><br />
								&nbsp;<em>-o</em>
					</td>
					<!--<td>VariantContextWriter</td> -->
					<td>stdout</td>
					<td>File to which variants should be written</td>
				</tr>
			<tr>
				<th colspan="4" id="row-divider">Optional Parameters</th>
			</tr>
				<tr>
					<td><a href="#--contamination_fraction_to_filter">--contamination_fraction_to_filter</a><br />
								&nbsp;<em>-contamination</em>
					</td>
					<!--<td>double</td> -->
					<td>0.0</td>
					<td>Fraction of contamination to aggressively remove</td>
				</tr>
				<tr>
					<td><a href="#--genotyping_mode">--genotyping_mode</a><br />
								&nbsp;<em>-gt_mode</em>
					</td>
					<!--<td>GenotypingOutputMode</td> -->
					<td>DISCOVERY</td>
					<td>Specifies how to determine the alternate alleles to use for genotyping</td>
				</tr>
				<tr>
					<td><a href="#--group">--group</a><br />
								&nbsp;<em>-G</em>
					</td>
					<!--<td>List[String]</td> -->
					<td>[StandardAnnotation, StandardHCAnnotation]</td>
					<td>One or more classes/groups of annotations to apply to variant calls</td>
				</tr>
				<tr>
					<td><a href="#--heterozygosity">--heterozygosity</a><br />
								&nbsp;<em>-hets</em>
					</td>
					<!--<td>Double</td> -->
					<td>0.001</td>
					<td>Heterozygosity value used to compute prior likelihoods for any locus</td>
				</tr>
				<tr>
					<td><a href="#--heterozygosity_stdev">--heterozygosity_stdev</a><br />
								&nbsp;<em>-heterozygosityStandardDeviation</em>
					</td>
					<!--<td>double</td> -->
					<td>0.01</td>
					<td>Standard deviation of eterozygosity for SNP and indel calling.</td>
				</tr>
				<tr>
					<td><a href="#--indel_heterozygosity">--indel_heterozygosity</a><br />
								&nbsp;<em>-indelHeterozygosity</em>
					</td>
					<!--<td>double</td> -->
					<td>1.25E-4</td>
					<td>Heterozygosity for indel calling</td>
				</tr>
				<tr>
					<td><a href="#--maxReadsInRegionPerSample">--maxReadsInRegionPerSample</a><br />
					</td>
					<!--<td>int</td> -->
					<td>10000</td>
					<td>Maximum reads in an active region</td>
				</tr>
				<tr>
					<td><a href="#--min_base_quality_score">--min_base_quality_score</a><br />
								&nbsp;<em>-mbq</em>
					</td>
					<!--<td>byte</td> -->
					<td>10</td>
					<td>Minimum base quality required to consider a base for calling</td>
				</tr>
				<tr>
					<td><a href="#--minReadsPerAlignmentStart">--minReadsPerAlignmentStart</a><br />
								&nbsp;<em>-minReadsPerAlignStart</em>
					</td>
					<!--<td>int</td> -->
					<td>10</td>
					<td>Minimum number of reads sharing the same alignment start for each genomic location in an active region</td>
				</tr>
				<tr>
					<td><a href="#--sample_name">--sample_name</a><br />
								&nbsp;<em>-sn</em>
					</td>
					<!--<td>String</td> -->
					<td>NA</td>
					<td>Name of single sample to use from a multi-sample bam</td>
				</tr>
				<tr>
					<td><a href="#--sample_ploidy">--sample_ploidy</a><br />
								&nbsp;<em>-ploidy</em>
					</td>
					<!--<td>int</td> -->
					<td>2</td>
					<td>Ploidy per sample. For pooled data, set to (Number of samples in each pool * Sample Ploidy).</td>
				</tr>
				<tr>
					<td><a href="#--standard_min_confidence_threshold_for_calling">--standard_min_confidence_threshold_for_calling</a><br />
								&nbsp;<em>-stand_call_conf</em>
					</td>
					<!--<td>double</td> -->
					<td>10.0</td>
					<td>The minimum phred-scaled confidence threshold at which variants should be called</td>
				</tr>
			<tr>
				<th colspan="4" id="row-divider">Optional Flags</th>
			</tr>
				<tr>
					<td><a href="#--annotateNDA">--annotateNDA</a><br />
								&nbsp;<em>-nda</em>
					</td>
					<!--<td>boolean</td> -->
					<td>false</td>
					<td>Annotate number of alleles observed</td>
				</tr>
				<tr>
					<td><a href="#--useNewAFCalculator">--useNewAFCalculator</a><br />
								&nbsp;<em>-newQual</em>
					</td>
					<!--<td>boolean</td> -->
					<td>false</td>
					<td>Use new AF model instead of the so-called exact model</td>
				</tr>
			<tr>
				<th colspan="4" id="row-divider">Advanced Inputs</th>
			</tr>
				<tr>
					<td><a href="#--activeRegionIn">--activeRegionIn</a><br />
								&nbsp;<em>-AR</em>
					</td>
					<!--<td>List[IntervalBinding[Feature]]</td> -->
					<td>NA</td>
					<td>Use this interval list file as the active regions to process</td>
				</tr>
				<tr>
					<td><a href="#--comp">--comp</a><br />
					</td>
					<!--<td>List[RodBinding[VariantContext]]</td> -->
					<td>[]</td>
					<td>Comparison VCF file</td>
				</tr>
			<tr>
				<th colspan="4" id="row-divider">Advanced Outputs</th>
			</tr>
				<tr>
					<td><a href="#--bamOutput">--bamOutput</a><br />
								&nbsp;<em>-bamout</em>
					</td>
					<!--<td>GATKSAMFileWriter</td> -->
					<td>NA</td>
					<td>File to which assembled haplotypes should be written</td>
				</tr>
			<tr>
				<th colspan="4" id="row-divider">Advanced Parameters</th>
			</tr>
				<tr>
					<td><a href="#--activeProbabilityThreshold">--activeProbabilityThreshold</a><br />
								&nbsp;<em>-ActProbThresh</em>
					</td>
					<!--<td>Double</td> -->
					<td>0.002</td>
					<td>Threshold for the probability of a profile state being active.</td>
				</tr>
				<tr>
					<td><a href="#--activeRegionExtension">--activeRegionExtension</a><br />
					</td>
					<!--<td>Integer</td> -->
					<td>NA</td>
					<td>The active region extension; if not provided defaults to Walker annotated default</td>
				</tr>
				<tr>
					<td><a href="#--activeRegionMaxSize">--activeRegionMaxSize</a><br />
					</td>
					<!--<td>Integer</td> -->
					<td>NA</td>
					<td>The active region maximum size; if not provided defaults to Walker annotated default</td>
				</tr>
				<tr>
					<td><a href="#--annotation">--annotation</a><br />
								&nbsp;<em>-A</em>
					</td>
					<!--<td>List[String]</td> -->
					<td>[]</td>
					<td>One or more specific annotations to apply to variant calls</td>
				</tr>
				<tr>
					<td><a href="#--bamWriterType">--bamWriterType</a><br />
					</td>
					<!--<td>Type</td> -->
					<td>CALLED_HAPLOTYPES</td>
					<td>Which haplotypes should be written to the BAM</td>
				</tr>
				<tr>
					<td><a href="#--bandPassSigma">--bandPassSigma</a><br />
					</td>
					<!--<td>Double</td> -->
					<td>NA</td>
					<td>The sigma of the band pass filter Gaussian kernel; if not provided defaults to Walker annotated default</td>
				</tr>
				<tr>
					<td><a href="#--contamination_fraction_per_sample_file">--contamination_fraction_per_sample_file</a><br />
								&nbsp;<em>-contaminationFile</em>
					</td>
					<!--<td>File</td> -->
					<td>NA</td>
					<td>Contamination per sample</td>
				</tr>
				<tr>
					<td><a href="#--emitRefConfidence">--emitRefConfidence</a><br />
								&nbsp;<em>-ERC</em>
					</td>
					<!--<td>ReferenceConfidenceMode</td> -->
					<td>false</td>
					<td>Mode for emitting reference confidence scores</td>
				</tr>
				<tr>
					<td><a href="#--excludeAnnotation">--excludeAnnotation</a><br />
								&nbsp;<em>-XA</em>
					</td>
					<!--<td>List[String]</td> -->
					<td>[]</td>
					<td>One or more specific annotations to exclude</td>
				</tr>
				<tr>
					<td><a href="#--gcpHMM">--gcpHMM</a><br />
					</td>
					<!--<td>int</td> -->
					<td>10</td>
					<td>Flat gap continuation penalty for use in the Pair HMM</td>
				</tr>
				<tr>
					<td><a href="#--GVCFGQBands">--GVCFGQBands</a><br />
								&nbsp;<em>-GQB</em>
					</td>
					<!--<td>List[Integer]</td> -->
					<td>[1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 70, 80, 90, 99]</td>
					<td>Exclusive upper bounds for reference confidence GQ bands (must be in [1, 100] and specified in increasing order)</td>
				</tr>
				<tr>
					<td><a href="#--indelSizeToEliminateInRefModel">--indelSizeToEliminateInRefModel</a><br />
								&nbsp;<em>-ERCIS</em>
					</td>
					<!--<td>int</td> -->
					<td>10</td>
					<td>The size of an indel to check for in the reference model</td>
				</tr>
				<tr>
					<td><a href="#--input_prior">--input_prior</a><br />
								&nbsp;<em>-inputPrior</em>
					</td>
					<!--<td>List[Double]</td> -->
					<td>[]</td>
					<td>Input prior for calls</td>
				</tr>
				<tr>
					<td><a href="#--kmerSize">--kmerSize</a><br />
					</td>
					<!--<td>List[Integer]</td> -->
					<td>[10, 25]</td>
					<td>Kmer size to use in the read threading assembler</td>
				</tr>
				<tr>
					<td><a href="#--max_alternate_alleles">--max_alternate_alleles</a><br />
								&nbsp;<em>-maxAltAlleles</em>
					</td>
					<!--<td>int</td> -->
					<td>6</td>
					<td>Maximum number of alternate alleles to genotype</td>
				</tr>
				<tr>
					<td><a href="#--max_genotype_count">--max_genotype_count</a><br />
								&nbsp;<em>-maxGT</em>
					</td>
					<!--<td>int</td> -->
					<td>1024</td>
					<td>Maximum number of genotypes to consider at any site</td>
				</tr>
				<tr>
					<td><a href="#--max_num_PL_values">--max_num_PL_values</a><br />
								&nbsp;<em>-maxNumPLValues</em>
					</td>
					<!--<td>int</td> -->
					<td>100</td>
					<td>Maximum number of PL values to output</td>
				</tr>
				<tr>
					<td><a href="#--maxNumHaplotypesInPopulation">--maxNumHaplotypesInPopulation</a><br />
					</td>
					<!--<td>int</td> -->
					<td>128</td>
					<td>Maximum number of haplotypes to consider for your population</td>
				</tr>
				<tr>
					<td><a href="#--maxReadsInMemoryPerSample">--maxReadsInMemoryPerSample</a><br />
					</td>
					<!--<td>int</td> -->
					<td>30000</td>
					<td>Maximum reads per sample given to traversal map() function</td>
				</tr>
				<tr>
					<td><a href="#--maxTotalReadsInMemory">--maxTotalReadsInMemory</a><br />
					</td>
					<!--<td>int</td> -->
					<td>10000000</td>
					<td>Maximum total reads given to traversal map() function</td>
				</tr>
				<tr>
					<td><a href="#--minDanglingBranchLength">--minDanglingBranchLength</a><br />
					</td>
					<!--<td>int</td> -->
					<td>4</td>
					<td>Minimum length of a dangling branch to attempt recovery</td>
				</tr>
				<tr>
					<td><a href="#--minPruning">--minPruning</a><br />
					</td>
					<!--<td>int</td> -->
					<td>2</td>
					<td>Minimum support to not prune paths in the graph</td>
				</tr>
				<tr>
					<td><a href="#--numPruningSamples">--numPruningSamples</a><br />
					</td>
					<!--<td>int</td> -->
					<td>1</td>
					<td>Number of samples that must pass the minPruning threshold</td>
				</tr>
				<tr>
					<td><a href="#--output_mode">--output_mode</a><br />
								&nbsp;<em>-out_mode</em>
					</td>
					<!--<td>OutputMode</td> -->
					<td>EMIT_VARIANTS_ONLY</td>
					<td>Which type of calls we should output</td>
				</tr>
				<tr>
					<td><a href="#--pcr_indel_model">--pcr_indel_model</a><br />
								&nbsp;<em>-pcrModel</em>
					</td>
					<!--<td>PCR_ERROR_MODEL</td> -->
					<td>CONSERVATIVE</td>
					<td>The PCR indel model to use</td>
				</tr>
				<tr>
					<td><a href="#--phredScaledGlobalReadMismappingRate">--phredScaledGlobalReadMismappingRate</a><br />
								&nbsp;<em>-globalMAPQ</em>
					</td>
					<!--<td>int</td> -->
					<td>45</td>
					<td>The global assumed mismapping rate for reads</td>
				</tr>
			<tr>
				<th colspan="4" id="row-divider">Advanced Flags</th>
			</tr>
				<tr>
					<td><a href="#--allowNonUniqueKmersInRef">--allowNonUniqueKmersInRef</a><br />
					</td>
					<!--<td>boolean</td> -->
					<td>false</td>
					<td>Allow graphs that have non-unique kmers in the reference</td>
				</tr>
				<tr>
					<td><a href="#--allSitePLs">--allSitePLs</a><br />
					</td>
					<!--<td>boolean</td> -->
					<td>false</td>
					<td>Annotate all sites with PLs</td>
				</tr>
				<tr>
					<td><a href="#--consensus">--consensus</a><br />
					</td>
					<!--<td>boolean</td> -->
					<td>false</td>
					<td>1000G consensus mode</td>
				</tr>
				<tr>
					<td><a href="#--debug">--debug</a><br />
					</td>
					<!--<td>boolean</td> -->
					<td>false</td>
					<td>Print out very verbose debug information about each triggering active region</td>
				</tr>
				<tr>
					<td><a href="#--disableOptimizations">--disableOptimizations</a><br />
					</td>
					<!--<td>boolean</td> -->
					<td>false</td>
					<td>Don't skip calculations in ActiveRegions with no variants</td>
				</tr>
				<tr>
					<td><a href="#--doNotRunPhysicalPhasing">--doNotRunPhysicalPhasing</a><br />
					</td>
					<!--<td>boolean</td> -->
					<td>false</td>
					<td>Disable physical phasing</td>
				</tr>
				<tr>
					<td><a href="#--dontIncreaseKmerSizesForCycles">--dontIncreaseKmerSizesForCycles</a><br />
					</td>
					<!--<td>boolean</td> -->
					<td>false</td>
					<td>Disable iterating over kmer sizes when graph cycles are detected</td>
				</tr>
				<tr>
					<td><a href="#--dontTrimActiveRegions">--dontTrimActiveRegions</a><br />
					</td>
					<!--<td>boolean</td> -->
					<td>false</td>
					<td>If specified, we will not trim down the active region from the full region (active + extension) to just the active interval for genotyping</td>
				</tr>
				<tr>
					<td><a href="#--dontUseSoftClippedBases">--dontUseSoftClippedBases</a><br />
					</td>
					<!--<td>boolean</td> -->
					<td>false</td>
					<td>Do not analyze soft clipped bases in the reads</td>
				</tr>
				<tr>
					<td><a href="#--emitDroppedReads">--emitDroppedReads</a><br />
								&nbsp;<em>-edr</em>
					</td>
					<!--<td>boolean</td> -->
					<td>false</td>
					<td>Emit reads that are dropped for filtering, trimming, realignment failure</td>
				</tr>
				<tr>
					<td><a href="#--forceActive">--forceActive</a><br />
					</td>
					<!--<td>boolean</td> -->
					<td>false</td>
					<td>If provided, all bases will be tagged as active</td>
				</tr>
				<tr>
					<td><a href="#--useAllelesTrigger">--useAllelesTrigger</a><br />
								&nbsp;<em>-allelesTrigger</em>
					</td>
					<!--<td>boolean</td> -->
					<td>false</td>
					<td>Use additional trigger on variants found in an external alleles file</td>
				</tr>
				<tr>
					<td><a href="#--useFilteredReadsForAnnotations">--useFilteredReadsForAnnotations</a><br />
					</td>
					<!--<td>boolean</td> -->
					<td>false</td>
					<td>Use the contamination-filtered read maps for the purposes of annotating variants</td>
				</tr>
						</tbody>
					</table>

							<h3>Argument details</h3>
							<p>Arguments in this list are specific to this tool. Keep in mind that other arguments are available that are shared with other tools (e.g. command-line GATK arguments); see Inherited arguments above.</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--activeProbabilityThreshold">--activeProbabilityThreshold </a>
			 / <small>-ActProbThresh</small>
		</h3>
		<p class="args">
			<b>Threshold for the probability of a profile state being active.</b><br />
			
		</p>
		<p>
			<span class="label label-info ">Double</span>
				&nbsp;<span class="label">0.002</span>
				&nbsp;<span class="label label-warning">[ [ 0</span>
				&nbsp;<span class="label label-warning">1 ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--activeRegionExtension">--activeRegionExtension </a>
			 / <small>-activeRegionExtension</small>
		</h3>
		<p class="args">
			<b>The active region extension; if not provided defaults to Walker annotated default</b><br />
			
		</p>
		<p>
			<span class="label label-info ">Integer</span>
				&nbsp;<span class="label">NA</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--activeRegionIn">--activeRegionIn </a>
			 / <small>-AR</small>
		</h3>
		<p class="args">
			<b>Use this interval list file as the active regions to process</b><br />
			
		</p>
		<p>
			<span class="label label-info ">List[IntervalBinding[Feature]]</span>
				&nbsp;<span class="label">NA</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--activeRegionMaxSize">--activeRegionMaxSize </a>
			 / <small>-activeRegionMaxSize</small>
		</h3>
		<p class="args">
			<b>The active region maximum size; if not provided defaults to Walker annotated default</b><br />
			
		</p>
		<p>
			<span class="label label-info ">Integer</span>
				&nbsp;<span class="label">NA</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--activeRegionOut">--activeRegionOut </a>
			 / <small>-ARO</small>
		</h3>
		<p class="args">
			<b>Output the active region to this IGV formatted file</b><br />
			If provided, this walker will write out its active and inactive regions
 to this file in the IGV formatted TAB deliminated output:

 http://www.broadinstitute.org/software/igv/IGV

 Intended to make debugging the active region calculations easier
		</p>
		<p>
			<span class="label label-info ">PrintStream</span>
				&nbsp;<span class="label">NA</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--activityProfileOut">--activityProfileOut </a>
			 / <small>-APO</small>
		</h3>
		<p class="args">
			<b>Output the raw activity profile results in IGV format</b><br />
			If provided, this walker will write out its activity profile (per bp probabilities of being active)
 to this file in the IGV formatted TAB deliminated output:

 http://www.broadinstitute.org/software/igv/IGV

 Intended to make debugging the activity profile calculations easier
		</p>
		<p>
			<span class="label label-info ">PrintStream</span>
				&nbsp;<span class="label">NA</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--alleles">--alleles </a>
			 / <small>-alleles</small>
		</h3>
		<p class="args">
			<b>Set of alleles to use in genotyping</b><br />
			When --genotyping_mode is set to GENOTYPE_GIVEN_ALLELES mode, the caller will genotype the samples using only the alleles provide in this callset. Note that this is not well tested in HaplotypeCaller, and is definitely not suitable for use with HaplotypeCaller in -ERC GVCF mode. In addition, it does not apply to MuTect2 at all.
		</p>
			<p>This argument supports reference-ordered data (ROD) files in the following formats: BCF2, VCF, VCF3</p>
		<p>
			<span class="label label-info ">RodBinding[VariantContext]</span>
				&nbsp;<span class="label">none</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--allowNonUniqueKmersInRef">--allowNonUniqueKmersInRef </a>
			 / <small>-allowNonUniqueKmersInRef</small>
		</h3>
		<p class="args">
			<b>Allow graphs that have non-unique kmers in the reference</b><br />
			By default, the program does not allow processing of reference sections that contain non-unique kmers. Disabling
 this check may cause problems in the assembly graph.
		</p>
		<p>
			<span class="label label-info ">boolean</span>
				&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--allSitePLs">--allSitePLs </a>
			 / <small>-allSitePLs</small>
		</h3>
		<p class="args">
			<b>Annotate all sites with PLs</b><br />
			Experimental argument FOR USE WITH UnifiedGenotyper ONLY: if SNP likelihood model
 is specified, and if EMIT_ALL_SITES output mode is set, when we set this argument then we
 will also emit PLs at all sites. This will give a measure of reference confidence and a
 measure of which alt alleles are more plausible (if any).
 WARNINGS:
 - This feature will inflate VCF file size considerably.
 - All SNP ALT alleles will be emitted with corresponding 10 PL values.
 - An error will be emitted if EMIT_ALL_SITES is not set, or if anything other than diploid
 SNP model is used
 - THIS WILL NOT WORK WITH HaplotypeCaller, GenotypeGVCFs or MuTect2! Use HaplotypeCaller with
 -ERC GVCF then GenotypeGVCFs instead. See the Best Practices documentation for more information.
		</p>
		<p>
			<span class="label label-info ">boolean</span>
				&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--annotateNDA">--annotateNDA </a>
			 / <small>-nda</small>
		</h3>
		<p class="args">
			<b>Annotate number of alleles observed</b><br />
			Depending on the value of the --max_alternate_alleles argument, we may genotype only a fraction of the alleles
 being sent on for genotyping. Using this argument instructs the genotyper to annotate (in the INFO field) the
 number of alternate alleles that were originally discovered (but not necessarily genotyped) at the site.
		</p>
		<p>
			<span class="label label-info ">boolean</span>
				&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--annotation">--annotation </a>
			 / <small>-A</small>
		</h3>
		<p class="args">
			<b>One or more specific annotations to apply to variant calls</b><br />
			Which annotations to add to the output VCF file. The single value 'none' removes the default annotations.
 See the VariantAnnotator -list argument to view available annotations.
		</p>
		<p>
			<span class="label label-info ">List[String]</span>
				&nbsp;<span class="label">[]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--bamOutput">--bamOutput </a>
			 / <small>-bamout</small>
		</h3>
		<p class="args">
			<b>File to which assembled haplotypes should be written</b><br />
			The assembled haplotypes and locally realigned reads will be written as BAM to this file if requested.  This is
 intended to be used only for troubleshooting purposes, in specific areas where you want to better understand
 why the caller is making specific calls. Turning on this mode may result in serious performance cost for the
 caller, so we do NOT recommend using this argument systematically as it will significantly increase runtime.

 The candidate haplotypes (called or all, depending on mode) are emitted as single reads covering the entire
 active region, coming from sample "HC" and a special read group called "ArtificialHaplotype". This will increase
 the pileup depth compared to what would be expected from the reads only, especially in complex regions.

 The reads are written out containing an "HC" tag (integer) that encodes which haplotype each read best matches
 according to the haplotype caller's likelihood calculation.  The use of this tag is primarily intended
 to allow good coloring of reads in IGV.  Simply go to "Color Alignments By > Tag" and enter "HC" to more
 easily see which reads go with these haplotype. You can also tell IGV to group reads by sample, which will
 separate the potential haplotypes from the reads. These features are illustrated in
 <a href="https://www.dropbox.com/s/xvy7sbxpf13x5bp/haplotypecaller%20bamout%20for%20docs.png">this screenshot</a>.

 Note that only reads that are actually informative about the haplotypes are emitted with the HC tag.
 By informative we mean that there's a meaningful difference in the likelihood of the read coming from one
 haplotype compared to the next best haplotype. When coloring reads by HC tag in IGV, uninformative reads will
 remain grey.

 Note also that not every input read is emitted to the bam in this mode. To include all trimmed, downsampled,
 filtered and uninformative reads, add the <code>--emitDroppedReads</code> argument.

 If multiple BAMs are passed as input to the tool (as is common for MuTect2), then they will be combined in the
 <code>-bamout</code> output and tagged with the appropriate sample names.
		</p>
		<p>
			<span class="label label-info ">GATKSAMFileWriter</span>
				&nbsp;<span class="label">NA</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--bamWriterType">--bamWriterType </a>
			 / <small>-bamWriterType</small>
		</h3>
		<p class="args">
			<b>Which haplotypes should be written to the BAM</b><br />
			The type of <code>-bamout</code> output we want to see. This determines whether HC will write out all of the haplotypes it
 considered (top 128 max) or just the ones that were selected as alleles and assigned to samples.
		</p>
			<p>
			The --bamWriterType argument is an enumerated type (Type), which can have one of the following values:
			<dl class="enum">
					<dt class="enum">ALL_POSSIBLE_HAPLOTYPES</dt>
					<dd class="enum">A mode that's for method developers.  Writes out all of the possible
 haplotypes considered, as well as reads aligned to each</dd>
					<dt class="enum">CALLED_HAPLOTYPES</dt>
					<dd class="enum">A mode for users.  Writes out the reads aligned only to the called
 haplotypes.  Useful to understand why the caller is calling what it is</dd>
			</dl>
			</p>
		<p>
			<span class="label label-info ">Type</span>
				&nbsp;<span class="label">CALLED_HAPLOTYPES</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--bandPassSigma">--bandPassSigma </a>
			 / <small>-bandPassSigma</small>
		</h3>
		<p class="args">
			<b>The sigma of the band pass filter Gaussian kernel; if not provided defaults to Walker annotated default</b><br />
			
		</p>
		<p>
			<span class="label label-info ">Double</span>
				&nbsp;<span class="label">NA</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--comp">--comp </a>
			 / <small>-comp</small>
		</h3>
		<p class="args">
			<b>Comparison VCF file</b><br />
			If a call overlaps with a record from the provided comp track, the INFO field will be annotated
 as such in the output with the track name (e.g. -comp:FOO will have 'FOO' in the INFO field). Records that are
 filtered in the comp track will be ignored. Note that 'dbSNP' has been special-cased (see the --dbsnp argument).
		</p>
			<p>This argument supports reference-ordered data (ROD) files in the following formats: BCF2, VCF, VCF3</p>
		<p>
			<span class="label label-info ">List[RodBinding[VariantContext]]</span>
				&nbsp;<span class="label">[]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--consensus">--consensus </a>
			 / <small>-consensus</small>
		</h3>
		<p class="args">
			<b>1000G consensus mode</b><br />
			This argument is specifically intended for 1000G consensus analysis mode. Setting this flag will inject all
 provided alleles to the assembly graph but will not forcibly genotype all of them.
		</p>
		<p>
			<span class="label label-info ">boolean</span>
				&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--contamination_fraction_per_sample_file">--contamination_fraction_per_sample_file </a>
			 / <small>-contaminationFile</small>
		</h3>
		<p class="args">
			<b>Contamination per sample</b><br />
			This argument specifies a file with two columns "sample" and "contamination" (separated by a tab)
  specifying the contamination level for those samples (where contamination is given as a
  decimal number, not an integer) per line. There should be no header. Samples that do not appear
  in this file will be processed with CONTAMINATION_FRACTION.
		</p>
		<p>
			<span class="label label-info ">File</span>
				&nbsp;<span class="label">NA</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--contamination_fraction_to_filter">--contamination_fraction_to_filter </a>
			 / <small>-contamination</small>
		</h3>
		<p class="args">
			<b>Fraction of contamination to aggressively remove</b><br />
			If this fraction is greater is than zero, the caller will aggressively attempt to remove
 contamination through biased down-sampling of reads (for all samples). Basically, it will ignore the
 contamination fraction of reads for each alternate allele.  So if the pileup contains N
 total bases, then we will try to remove (N * contamination fraction) bases for each alternate
 allele.
		</p>
		<p>
			<span class="label label-info ">double</span>
				&nbsp;<span class="label">0.0</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--dbsnp">--dbsnp </a>
			 / <small>-D</small>
		</h3>
		<p class="args">
			<b>dbSNP file</b><br />
			rsIDs from this file are used to populate the ID column of the output. Also, the DB INFO flag will be set when appropriate.
 dbSNP is not used in any way for the calculations themselves.
		</p>
			<p>This argument supports reference-ordered data (ROD) files in the following formats: BCF2, VCF, VCF3</p>
		<p>
			<span class="label label-info ">RodBinding[VariantContext]</span>
				&nbsp;<span class="label">none</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--debug">--debug </a>
			 / <small>-debug</small>
		</h3>
		<p class="args">
			<b>Print out very verbose debug information about each triggering active region</b><br />
			
		</p>
		<p>
			<span class="label label-info ">boolean</span>
				&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--disableOptimizations">--disableOptimizations </a>
			 / <small>-disableOptimizations</small>
		</h3>
		<p class="args">
			<b>Don't skip calculations in ActiveRegions with no variants</b><br />
			If set, certain "early exit" optimizations in HaplotypeCaller, which aim to save compute and time by skipping
 calculations if an ActiveRegion is determined to contain no variants, will be disabled. This is most likely to be
 useful if you're using the <code>-bamout</code> argument to examine the placement of reads following reassembly
 and are interested in seeing the mapping of reads in regions with no variations. Setting the <code>-forceActive</code>
 and <code>-dontTrimActiveRegions</code> flags may also be helpful.
		</p>
		<p>
			<span class="label label-info ">boolean</span>
				&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--doNotRunPhysicalPhasing">--doNotRunPhysicalPhasing </a>
			 / <small>-doNotRunPhysicalPhasing</small>
		</h3>
		<p class="args">
			<b>Disable physical phasing</b><br />
			As of GATK 3.3, HaplotypeCaller outputs physical (read-based) information (see version 3.3 release notes and documentation for details). This argument disables that behavior.
		</p>
		<p>
			<span class="label label-info ">boolean</span>
				&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--dontIncreaseKmerSizesForCycles">--dontIncreaseKmerSizesForCycles </a>
			 / <small>-dontIncreaseKmerSizesForCycles</small>
		</h3>
		<p class="args">
			<b>Disable iterating over kmer sizes when graph cycles are detected</b><br />
			When graph cycles are detected, the normal behavior is to increase kmer sizes iteratively until the cycles are
 resolved. Disabling this behavior may cause the program to give up on assembling the ActiveRegion.
		</p>
		<p>
			<span class="label label-info ">boolean</span>
				&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--dontTrimActiveRegions">--dontTrimActiveRegions </a>
			 / <small>-dontTrimActiveRegions</small>
		</h3>
		<p class="args">
			<b>If specified, we will not trim down the active region from the full region (active + extension) to just the active interval for genotyping</b><br />
			
		</p>
		<p>
			<span class="label label-info ">boolean</span>
				&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--dontUseSoftClippedBases">--dontUseSoftClippedBases </a>
			 / <small>-dontUseSoftClippedBases</small>
		</h3>
		<p class="args">
			<b>Do not analyze soft clipped bases in the reads</b><br />
			
		</p>
		<p>
			<span class="label label-info ">boolean</span>
				&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--emitDroppedReads">--emitDroppedReads </a>
			 / <small>-edr</small>
		</h3>
		<p class="args">
			<b>Emit reads that are dropped for filtering, trimming, realignment failure</b><br />
			Determines whether dropped reads will be tracked and emitted when <code>-bamout</code> is specified. Use this in combination
 with a specific interval of interest to avoid accumulating a large number of reads in the <code>-bamout</code> file.
		</p>
		<p>
			<span class="label label-info ">boolean</span>
				&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--emitRefConfidence">--emitRefConfidence </a>
			 / <small>-ERC</small>
		</h3>
		<p class="args">
			<b>Mode for emitting reference confidence scores</b><br />
			Records whether the trimming intervals are going to be used to emit reference confidence, {@code true},
 or regular HC output {@code false}.
		</p>
			<p>
			The --emitRefConfidence argument is an enumerated type (ReferenceConfidenceMode), which can have one of the following values:
			<dl class="enum">
					<dt class="enum">NONE</dt>
					<dd class="enum">Regular calling without emitting reference confidence calls.</dd>
					<dt class="enum">BP_RESOLUTION</dt>
					<dd class="enum">Reference model emitted site by site.</dd>
					<dt class="enum">GVCF</dt>
					<dd class="enum">Reference model emitted with condensed non-variant blocks, i.e. the GVCF format.</dd>
			</dl>
			</p>
		<p>
			<span class="label label-info ">ReferenceConfidenceMode</span>
				&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--excludeAnnotation">--excludeAnnotation </a>
			 / <small>-XA</small>
		</h3>
		<p class="args">
			<b>One or more specific annotations to exclude</b><br />
			Which annotations to exclude from output in the VCF file.  Note that this argument has higher priority than the
 -A or -G arguments, so these annotations will be excluded even if they are explicitly included with the other
 options. When HaplotypeCaller is run with -ERC GVCF or -ERC BP_RESOLUTION, some annotations are excluded from the
 output by default because they will only be meaningful once they have been recalculated by GenotypeGVCFs. As
 of version 3.3 this concerns ChromosomeCounts, FisherStrand, StrandOddsRatio and QualByDepth.
		</p>
		<p>
			<span class="label label-info ">List[String]</span>
				&nbsp;<span class="label">[]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--forceActive">--forceActive </a>
			 / <small>-forceActive</small>
		</h3>
		<p class="args">
			<b>If provided, all bases will be tagged as active</b><br />
			For the active region walker to treat all bases as active.  Useful for debugging when you want to force something like
 the HaplotypeCaller to process a specific interval you provide the GATK
		</p>
		<p>
			<span class="label label-info ">boolean</span>
				&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--gcpHMM">--gcpHMM </a>
			 / <small>-gcpHMM</small>
		</h3>
		<p class="args">
			<b>Flat gap continuation penalty for use in the Pair HMM</b><br />
			
		</p>
		<p>
			<span class="label label-info ">int</span>
				&nbsp;<span class="label">10</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--genotyping_mode">--genotyping_mode </a>
			 / <small>-gt_mode</small>
		</h3>
		<p class="args">
			<b>Specifies how to determine the alternate alleles to use for genotyping</b><br />
			
		</p>
			<p>
			The --genotyping_mode argument is an enumerated type (GenotypingOutputMode), which can have one of the following values:
			<dl class="enum">
					<dt class="enum">DISCOVERY</dt>
					<dd class="enum">The genotyper will choose the most likely alternate allele</dd>
					<dt class="enum">GENOTYPE_GIVEN_ALLELES</dt>
					<dd class="enum">Only the alleles passed by the user should be considered.</dd>
			</dl>
			</p>
		<p>
			<span class="label label-info ">GenotypingOutputMode</span>
				&nbsp;<span class="label">DISCOVERY</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--graphOutput">--graphOutput </a>
			 / <small>-graph</small>
		</h3>
		<p class="args">
			<b>Write debug assembly graph information to this file</b><br />
			This argument is meant for debugging and is not immediately useful for normal analysis use.
		</p>
		<p>
			<span class="label label-info ">PrintStream</span>
				&nbsp;<span class="label">NA</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--group">--group </a>
			 / <small>-G</small>
		</h3>
		<p class="args">
			<b>One or more classes/groups of annotations to apply to variant calls</b><br />
			Which groups of annotations to add to the output VCF file. The single value 'none' removes the default group. See
 the VariantAnnotator -list argument to view available groups. Note that this usage is not recommended because
 it obscures the specific requirements of individual annotations. Any requirements that are not met (e.g. failing
 to provide a pedigree file for a pedigree-based annotation) may cause the run to fail.
		</p>
		<p>
			<span class="label label-info ">List[String]</span>
				&nbsp;<span class="label">[StandardAnnotation, StandardHCAnnotation]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--GVCFGQBands">--GVCFGQBands </a>
			 / <small>-GQB</small>
		</h3>
		<p class="args">
			<b>Exclusive upper bounds for reference confidence GQ bands (must be in [1, 100] and specified in increasing order)</b><br />
			When HC is run in reference confidence mode with banding compression enabled (-ERC GVCF), homozygous-reference
 sites are compressed into bands of similar genotype quality (GQ) that are emitted as a single VCF record. See
 the FAQ documentation for more details about the GVCF format.

 This argument allows you to set the GQ bands. HC expects a list of strictly increasing GQ values
 that will act as exclusive upper bounds for the GQ bands. To pass multiple values,
 you provide them one by one with the argument, as in `-GQB 10 -GQB 20 -GQB 30` and so on
 (this would set the GQ bands to be `[0, 10), [10, 20), [20, 30)` and so on, for example).
 Note that GQ values are capped at 99 in the GATK, so values must be integers in [1, 100].
 If the last value is strictly less than 100, the last GQ band will start at that value (inclusive)
 and end at 100 (exclusive).
		</p>
		<p>
			<span class="label label-info ">List[Integer]</span>
				&nbsp;<span class="label">[1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 70, 80, 90, 99]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--heterozygosity">--heterozygosity </a>
			 / <small>-hets</small>
		</h3>
		<p class="args">
			<b>Heterozygosity value used to compute prior likelihoods for any locus</b><br />
			The expected heterozygosity value used to compute prior probability that a locus is non-reference. See
 https://software.broadinstitute.org/gatk/documentation/article?id=8603 for more details.
		</p>
		<p>
			<span class="label label-info ">Double</span>
				&nbsp;<span class="label">0.001</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--heterozygosity_stdev">--heterozygosity_stdev </a>
			 / <small>-heterozygosityStandardDeviation</small>
		</h3>
		<p class="args">
			<b>Standard deviation of eterozygosity for SNP and indel calling.</b><br />
			The standard deviation of the distribution of alt allele fractions. The above heterozygosity parameters give
 the *mean* of this distribution; this parameter gives its spread.
		</p>
		<p>
			<span class="label label-info ">double</span>
				&nbsp;<span class="label">0.01</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--indel_heterozygosity">--indel_heterozygosity </a>
			 / <small>-indelHeterozygosity</small>
		</h3>
		<p class="args">
			<b>Heterozygosity for indel calling</b><br />
			This argument informs the prior probability of having an indel at a site.
		</p>
		<p>
			<span class="label label-info ">double</span>
				&nbsp;<span class="label">1.25E-4</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--indelSizeToEliminateInRefModel">--indelSizeToEliminateInRefModel </a>
			 / <small>-ERCIS</small>
		</h3>
		<p class="args">
			<b>The size of an indel to check for in the reference model</b><br />
			This parameter determines the maximum size of an indel considered as potentially segregating in the
 reference model.  It is used to eliminate reads from being indel informative at a site, and determines
 by that mechanism the certainty in the reference base.  Conceptually, setting this parameter to
 X means that each informative read is consistent with any indel of size < X being present at a specific
 position in the genome, given its alignment to the reference.
		</p>
		<p>
			<span class="label label-info ">int</span>
				&nbsp;<span class="label">10</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--input_prior">--input_prior </a>
			 / <small>-inputPrior</small>
		</h3>
		<p class="args">
			<b>Input prior for calls</b><br />
			By default, the prior specified with the argument --heterozygosity/-hets is used for variant discovery at a
 particular locus, using an infinite sites model (see e.g. Waterson, 1975 or Tajima, 1996). This model asserts that
 the probability of having a population of k variant sites in N chromosomes is proportional to theta/k, for 1=1:N.
 However, there are instances where using this prior might not be desirable, e.g. for population studies where prior
 might not be appropriate, as for example when the ancestral status of the reference allele is not known.

 This argument allows you to manually specify a list of probabilities for each AC>1 to be used as
 priors for genotyping, with the following restrictions: only diploid calls are supported; you must specify 2 *
 N values where N is the number of samples; probability values must be positive and specified in Double format,
 in linear space (not log10 space nor Phred-scale); and all values must sume to 1.

 For completely flat priors,  specify the same value (=1/(2*N+1)) 2*N times, e.g.
      -inputPrior 0.33 -inputPrior 0.33
 for the single-sample diploid case.
		</p>
		<p>
			<span class="label label-info ">List[Double]</span>
				&nbsp;<span class="label">[]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--kmerSize">--kmerSize </a>
			 / <small>-kmerSize</small>
		</h3>
		<p class="args">
			<b>Kmer size to use in the read threading assembler</b><br />
			Multiple kmer sizes can be specified, using e.g. `-kmerSize 10 -kmerSize 25`.
		</p>
		<p>
			<span class="label label-info ">List[Integer]</span>
				&nbsp;<span class="label">[10, 25]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--max_alternate_alleles">--max_alternate_alleles </a>
			 / <small>-maxAltAlleles</small>
		</h3>
		<p class="args">
			<b>Maximum number of alternate alleles to genotype</b><br />
			If there are more than this number of alternate alleles presented to the genotyper (either through discovery or
 GENOTYPE_GIVEN_ALLELES), then only this many alleles will be used.  Note that genotyping sites with many
 alternate alleles is both CPU and memory intensive and it scales exponentially based on the number of alternate
 alleles.  Unless there is a good reason to change the default value, we highly recommend that you not play around
 with this parameter.

 See also {@link #MAX_GENOTYPE_COUNT}.
		</p>
		<p>
			<span class="label label-info ">int</span>
				&nbsp;<span class="label">6</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--max_genotype_count">--max_genotype_count </a>
			 / <small>-maxGT</small>
		</h3>
		<p class="args">
			<b>Maximum number of genotypes to consider at any site</b><br />
			If there are more than this number of genotypes at a locus presented to the genotyper, then only this many
 genotypes will be used. This is intended to deal with sites where the combination of high ploidy and high alt
 allele count can lead to an explosion in the number of possible genotypes, with extreme adverse effects on
 runtime performance.

 How does it work? The possible genotypes are simply different ways of partitioning alleles given a specific
 ploidy assumption. Therefore, we remove genotypes from consideration by removing alternate alleles that are the
 least well supported. The estimate of allele support is based on the ranking of the candidate haplotypes coming
 out of the graph building step. Note however that the reference allele is always kept.

 The maximum number of alternative alleles used in the genotyping step will be the lesser of the two:
 1. the largest number of alt alleles, given ploidy, that yields a genotype count no higher than {@link #MAX_GENOTYPE_COUNT}
 2. the value of {@link #MAX_ALTERNATE_ALLELES}

 As noted above, genotyping sites with large genotype counts is both CPU and memory intensive. Unless you have
 a good reason to change the default value, we highly recommend that you not play around with this parameter.

 See also {@link #MAX_ALTERNATE_ALLELES}.
		</p>
		<p>
			<span class="label label-info ">int</span>
				&nbsp;<span class="label">1024</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--max_num_PL_values">--max_num_PL_values </a>
			 / <small>-maxNumPLValues</small>
		</h3>
		<p class="args">
			<b>Maximum number of PL values to output</b><br />
			Determines the maximum number of PL values that will be logged in the output.  If the number of genotypes
 (which is determined by the ploidy and the number of alleles) exceeds the value provided by this argument,
 then output of all of the PL values will be suppressed.
		</p>
		<p>
			<span class="label label-info ">int</span>
				&nbsp;<span class="label">100</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--maxNumHaplotypesInPopulation">--maxNumHaplotypesInPopulation </a>
			 / <small>-maxNumHaplotypesInPopulation</small>
		</h3>
		<p class="args">
			<b>Maximum number of haplotypes to consider for your population</b><br />
			The assembly graph can be quite complex, and could imply a very large number of possible haplotypes.  Each haplotype
 considered requires N PairHMM evaluations if there are N reads across all samples.  In order to control the
 run of the haplotype caller we only take maxNumHaplotypesInPopulation paths from the graph, in order of their
 weights, no matter how many paths are possible to generate from the graph.  Putting this number too low
 will result in dropping true variation because paths that include the real variant are not even considered.
 You can consider increasing this number when calling organisms with high heterozygosity.
		</p>
		<p>
			<span class="label label-info ">int</span>
				&nbsp;<span class="label">128</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--maxReadsInMemoryPerSample">--maxReadsInMemoryPerSample </a>
			 / <small>-maxReadsInMemoryPerSample</small>
		</h3>
		<p class="args">
			<b>Maximum reads per sample given to traversal map() function</b><br />
			What is the maximum number of reads we're willing to hold in memory per sample
 during the traversal?  This limits our exposure to unusually large amounts
 of coverage in the engine.
		</p>
		<p>
			<span class="label label-info ">int</span>
				&nbsp;<span class="label">30000</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--maxReadsInRegionPerSample">--maxReadsInRegionPerSample </a>
			 / <small>-maxReadsInRegionPerSample</small>
		</h3>
		<p class="args">
			<b>Maximum reads in an active region</b><br />
			When downsampling, level the coverage of the reads in each sample to no more than maxReadsInRegionPerSample reads,
 not reducing coverage at any read start to less than minReadsPerAlignmentStart
		</p>
		<p>
			<span class="label label-info ">int</span>
				&nbsp;<span class="label">10000</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--maxTotalReadsInMemory">--maxTotalReadsInMemory </a>
			 / <small>-maxTotalReadsInMemory</small>
		</h3>
		<p class="args">
			<b>Maximum total reads given to traversal map() function</b><br />
			What is the maximum number of reads we're willing to hold in memory per sample
 during the traversal?  This limits our exposure to unusually large amounts
 of coverage in the engine.
		</p>
		<p>
			<span class="label label-info ">int</span>
				&nbsp;<span class="label">10000000</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--min_base_quality_score">--min_base_quality_score </a>
			 / <small>-mbq</small>
		</h3>
		<p class="args">
			<b>Minimum base quality required to consider a base for calling</b><br />
			Bases with a quality below this threshold will not be used for calling.
		</p>
		<p>
			<span class="label label-info ">byte</span>
				&nbsp;<span class="label">10</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--minDanglingBranchLength">--minDanglingBranchLength </a>
			 / <small>-minDanglingBranchLength</small>
		</h3>
		<p class="args">
			<b>Minimum length of a dangling branch to attempt recovery</b><br />
			When constructing the assembly graph we are often left with "dangling" branches.  The assembly engine attempts to rescue these branches
 by merging them back into the main graph.  This argument describes the minimum length of a dangling branch needed for the engine to
 try to rescue it.  A smaller number here will lead to higher sensitivity to real variation but also to a higher number of false positives.
		</p>
		<p>
			<span class="label label-info ">int</span>
				&nbsp;<span class="label">4</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--minPruning">--minPruning </a>
			 / <small>-minPruning</small>
		</h3>
		<p class="args">
			<b>Minimum support to not prune paths in the graph</b><br />
			Paths with fewer supporting kmers than the specified threshold will be pruned from the graph.

 Be aware that this argument can dramatically affect the results of variant calling and should only be used with great caution.
 Using a prune factor of 1 (or below) will prevent any pruning from the graph, which is generally not ideal; it can make the
 calling much slower and even less accurate (because it can prevent effective merging of "tails" in the graph).  Higher values
 tend to make the calling much faster, but also lowers the sensitivity of the results (because it ultimately requires higher
 depth to produce calls).
		</p>
		<p>
			<span class="label label-info ">int</span>
				&nbsp;<span class="label">2</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--minReadsPerAlignmentStart">--minReadsPerAlignmentStart </a>
			 / <small>-minReadsPerAlignStart</small>
		</h3>
		<p class="args">
			<b>Minimum number of reads sharing the same alignment start for each genomic location in an active region</b><br />
			
		</p>
		<p>
			<span class="label label-info ">int</span>
				&nbsp;<span class="label">10</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--numPruningSamples">--numPruningSamples </a>
			 / <small>-numPruningSamples</small>
		</h3>
		<p class="args">
			<b>Number of samples that must pass the minPruning threshold</b><br />
			If fewer samples than the specified number pass the minPruning threshold for a given path, that path will be eliminated from the graph.
		</p>
		<p>
			<span class="label label-info ">int</span>
				&nbsp;<span class="label">1</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--out">--out </a>
			 / <small>-o</small>
		</h3>
		<p class="args">
			<b>File to which variants should be written</b><br />
			A raw, unfiltered, highly sensitive callset in VCF format.
		</p>
		<p>
			<span class="label label-info ">VariantContextWriter</span>
				&nbsp;<span class="label">stdout</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--output_mode">--output_mode </a>
			 / <small>-out_mode</small>
		</h3>
		<p class="args">
			<b>Which type of calls we should output</b><br />
			Experimental argument FOR USE WITH UnifiedGenotyper ONLY. When using HaplotypeCaller, use -ERC
 instead. When using GenotypeGVCFs, see -allSites.
		</p>
			<p>
			The --output_mode argument is an enumerated type (OutputMode), which can have one of the following values:
			<dl class="enum">
					<dt class="enum">EMIT_VARIANTS_ONLY</dt>
					<dd class="enum">produces calls only at variant sites</dd>
					<dt class="enum">EMIT_ALL_CONFIDENT_SITES</dt>
					<dd class="enum">produces calls at variant sites and confident reference sites</dd>
					<dt class="enum">EMIT_ALL_SITES</dt>
					<dd class="enum">produces calls at any callable site regardless of confidence; this argument is intended only for point
 mutations (SNPs) in DISCOVERY mode or generally when running in GENOTYPE_GIVEN_ALLELES mode; it will by
 no means produce a comprehensive set of indels in DISCOVERY mode</dd>
			</dl>
			</p>
		<p>
			<span class="label label-info ">OutputMode</span>
				&nbsp;<span class="label">EMIT_VARIANTS_ONLY</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--pcr_indel_model">--pcr_indel_model </a>
			 / <small>-pcrModel</small>
		</h3>
		<p class="args">
			<b>The PCR indel model to use</b><br />
			When calculating the likelihood of variants, we can try to correct for PCR errors that cause indel artifacts.
 The correction is based on the reference context, and acts specifically around repetitive sequences that tend
 to cause PCR errors). The variant likelihoods are penalized in increasing scale as the context around a
 putative indel is more repetitive (e.g. long homopolymer). The correction can be disabling by specifying
 '-pcrModel NONE'; in that case the default base insertion/deletion qualities will be used (or taken from the
 read if generated through the BaseRecalibrator). <b>VERY IMPORTANT: when using PCR-free sequencing data we
 definitely recommend setting this argument to NONE</b>.
		</p>
			<p>
			The --pcr_indel_model argument is an enumerated type (PCR_ERROR_MODEL), which can have one of the following values:
			<dl class="enum">
					<dt class="enum">NONE</dt>
					<dd class="enum">no specialized PCR error model will be applied; if base insertion/deletion qualities are present they will be used</dd>
					<dt class="enum">HOSTILE</dt>
					<dd class="enum">a most aggressive model will be applied that sacrifices true positives in order to remove more false positives</dd>
					<dt class="enum">AGGRESSIVE</dt>
					<dd class="enum">a more aggressive model will be applied that sacrifices true positives in order to remove more false positives</dd>
					<dt class="enum">CONSERVATIVE</dt>
					<dd class="enum">a less aggressive model will be applied that tries to maintain a high true positive rate at the expense of allowing more false positives</dd>
			</dl>
			</p>
		<p>
			<span class="label label-info ">PCR_ERROR_MODEL</span>
				&nbsp;<span class="label">CONSERVATIVE</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--phredScaledGlobalReadMismappingRate">--phredScaledGlobalReadMismappingRate </a>
			 / <small>-globalMAPQ</small>
		</h3>
		<p class="args">
			<b>The global assumed mismapping rate for reads</b><br />
			The phredScaledGlobalReadMismappingRate reflects the average global mismapping rate of all reads, regardless of their
 mapping quality.  This term effects the probability that a read originated from the reference haplotype, regardless of
 its edit distance from the reference, in that the read could have originated from the reference haplotype but
 from another location in the genome.  Suppose a read has many mismatches from the reference, say like 5, but
 has a very high mapping quality of 60.  Without this parameter, the read would contribute 5 * Q30 evidence
 in favor of its 5 mismatch haplotype compared to reference, potentially enough to make a call off that single
 read for all of these events.  With this parameter set to Q30, though, the maximum evidence against any haplotype
 that this (and any) read could contribute is Q30.

 Set this term to any negative number to turn off the global mapping rate.
		</p>
		<p>
			<span class="label label-info ">int</span>
				&nbsp;<span class="label">45</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--sample_name">--sample_name </a>
			 / <small>-sn</small>
		</h3>
		<p class="args">
			<b>Name of single sample to use from a multi-sample bam</b><br />
			You can use this argument to specify that HC should process a single sample out of a multisample BAM file. This
 is especially useful if your samples are all in the same file but you need to run them individually through HC
 in -ERC GVC mode (which is the recommended usage). Note that the name is case-sensitive.
		</p>
		<p>
			<span class="label label-info ">String</span>
				&nbsp;<span class="label">NA</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--sample_ploidy">--sample_ploidy </a>
			 / <small>-ploidy</small>
		</h3>
		<p class="args">
			<b>Ploidy per sample. For pooled data, set to (Number of samples in each pool * Sample Ploidy).</b><br />
			Sample ploidy - equivalent to number of chromosome copies per pool. For pooled experiments this should be set to
   the number of samples in pool multiplied by individual sample ploidy.
		</p>
		<p>
			<span class="label label-info ">int</span>
				&nbsp;<span class="label">2</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--standard_min_confidence_threshold_for_calling">--standard_min_confidence_threshold_for_calling </a>
			 / <small>-stand_call_conf</small>
		</h3>
		<p class="args">
			<b>The minimum phred-scaled confidence threshold at which variants should be called</b><br />
			The minimum phred-scaled Qscore threshold to separate high confidence from low confidence calls. Only genotypes with
 confidence >= this threshold are emitted as called sites. A reasonable threshold is 30 for high-pass calling (this
 is the default).
		</p>
		<p>
			<span class="label label-info ">double</span>
				&nbsp;<span class="label">10.0</span>
				&nbsp;<span class="label label-warning">[ [ -</span>
				&nbsp;<span class="label label-warning"> ] ]</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--useAllelesTrigger">--useAllelesTrigger </a>
			 / <small>-allelesTrigger</small>
		</h3>
		<p class="args">
			<b>Use additional trigger on variants found in an external alleles file</b><br />
			
		</p>
		<p>
			<span class="label label-info ">boolean</span>
				&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--useFilteredReadsForAnnotations">--useFilteredReadsForAnnotations </a>
			 / <small>-useFilteredReadsForAnnotations</small>
		</h3>
		<p class="args">
			<b>Use the contamination-filtered read maps for the purposes of annotating variants</b><br />
			
		</p>
		<p>
			<span class="label label-info ">boolean</span>
				&nbsp;<span class="label">false</span>
		</p>
		<hr style="border-bottom: dotted 1px #C0C0C0;" />
		<h3><a name="--useNewAFCalculator">--useNewAFCalculator </a>
			 / <small>-newQual</small>
		</h3>
		<p class="args">
			<b>Use new AF model instead of the so-called exact model</b><br />
			This activates a model for calculating QUAL that was introduced in version 3.7 (November 2016). We expect this
 model will become the default in future versions.
		</p>
		<p>
			<span class="label label-info ">boolean</span>
				&nbsp;<span class="label">false</span>
		</p>

        <hr>


	</div>

</div>

